---
title: "The microbiota of the proximal and distal human colon"
csl: gastro.csl
fontsize: 12pt
output:
  word_document: default
  pdf_document:
    includes: null
    keep_tex: yes
  html_document: default
header-includes:
- \usepackage{setspace}
- \doublespacing
- \usepackage{lineno}
- \linenumbers
- \renewcommand{\familydefault}{\sfdefault}
- \usepackage{graphicx}
geometry: margin=1.0in
bibliography: references.bib
---
\vspace{35mm}

Running title: The microbiota of the proximal and distal human colon

\vspace{35mm}


Kaitlin J. Flynn^1^, Charles C. Koumpouras^1^, Mack T. Ruffin IV^2^, D. Kim Turgeon^3$\dagger$^, and Patrick D. Schloss^1$\dagger$^

\vspace{35mm}

$\dagger$ Corresponding authors: kturgeon@umich.edu and pschloss@umich.edu


1\. Department of Microbiology and Immunology, University of Michigan Medical School, Ann Arbor, Michigan 48109

2\. Pennsylvania State University, Hershey, Pennsylvania ??

3\. Department of Internal Medicine, Division of Gastroenterology, University of Michigan Medical School, Ann Arbor, Michigan


\newpage

```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE}
knitr::opts_chunk$set("tidy" = TRUE)
knitr::opts_chunk$set("echo" = FALSE)
knitr::opts_chunk$set("eval" = TRUE)
knitr::opts_chunk$set("warning" = FALSE)
knitr::opts_chunk$set("cache" = FALSE)


#knitr::knit_hooks$set(inline=inline_hook)
```

```{r dependencies, echo=FALSE, warning = FALSE, message=FALSE}
pack_used <- c('randomForest','ggplot2', 'pROC', 'knitr','dplyr','AUCRF', 'tidyr', 'caret', 'RColorBrewer', 'reshape2', 'wesanderson')
for (dep in pack_used){
  if (dep %in% installed.packages()[,"Package"] == FALSE){
    install.packages(as.character(dep), repos = 'http://cran.us.r-project.org', 
                     quiet=TRUE);
  }
  library(dep, verbose=FALSE, character.only=TRUE)
}
```

```{r datasets, echo=FALSE}
meta <- '../data/raw/kws_metadata.tsv'
shared <- '../data/mothur/kws_final.an.shared'
tax <- '../data/mothur/kws_final.an.cons.taxonomy'
subsample <- read.table(file='../data/mothur/kws_final.an.0.03.subsample.shared', header=T)

meta_file <- read.table(file='../data/raw/kws_metadata.tsv', header = T)
shared_file <- read.table(file='../data/mothur/kws_final.an.shared', sep = '\t', header=T, row.names=2)
tax_file <- read.table(file='../data/mothur/kws_final.an.cons.taxonomy', sep = '\t', header=T, row.names=1)
shared_meta <- merge(meta_file, shared_file, by.x='group', by.y='row.names')

simps <- read.table(file='../data/mothur/kws_final.an.groups.summary', header = T)
simpmeta <- merge(meta_file, simps)

tyc <- read.table(file ="../data/mothur/kws_final.an.summary", sep = '\t', header = T, row.names=NULL)

```

```{r otu_barplot, echo=FALSE}
source('../code/Sum_OTU_by_Tax.R')
source('../code/sum_shared.R')

#use this code to assign phyla to each OTU in the shared file 
sub <- '../data/mothur/kws_final.an.0.03.subsample.shared'
shared_phyla <- get_tax_level_shared(sub, tax, 2)
phyla_met <- merge(meta_file, shared_phyla, by.x='group', by.y='row.names')

#try to get the df organized to work as a boxplot - no median calculation 
phyla_test <- phyla_met[, c("location","Firmicutes","Bacteroidetes","Proteobacteria","Verrucomicrobia","Actinobacteria","Fusobacteria")]
subsampled_to <- 4231
RA <- function(x) 100*x/subsampled_to

phyla_RA <- data.frame(phyla_test[1], apply(phyla_test[2:ncol(phyla_test)],2, RA))
phylaRAnames <- colnames(phyla_RA[,1:7])
phyla_RAmelt <- melt(phyla_RA[, phylaRAnames], id.vars=1)

```

```{r thetayc, echo=FALSE}
tyc <- read.table("../data/mothur/kws_final.an.summary", sep = '\t', header = T, row.names=NULL)

#separate column values for comparisons
tyc <- separate(tyc, label, into= c('pt1', 'samp1'), sep="-", remove=F)
tyc <- separate(tyc, comparison, into= c('pt2', 'samp2'), sep="-", remove=F)
tyc <- subset(tyc, select = -c(row.names, X))

tyc <- subset(tyc, pt1==pt2)
tyc <- unite_(tyc, "match", from=c('samp1', 'samp2'), sep="_", remove = F)

stooltyc <- subset(tyc, match=='LB_RB' | match== 'LS_RS')
distalandproximaltyc <- subset(tyc, match=='LB_LS' | match== 'RB_RS')
lvsr <- rbind(stooltyc, distalandproximaltyc)

exittyc <- subset(tyc, samp2 == 'SS')

pvalues <- c()

Atyc <- subset(tyc, match=='RB_RS' | match=='LS_RS')
pvalues <- c(pvalues, wilcox.test(thetayc~match, data=Atyc, paired=T)$p.value)

btyc <- subset(tyc, match=='RB_RS'| match=='LB_RB')
btyc <- btyc[-25,]
pvalues <- c(pvalues, wilcox.test(thetayc~match, data=btyc, paired=T)$p.value)

ctyc <- subset(tyc, match=='RB_RS'| match=='LB_LS')
ctyc <- ctyc[-25,]
pvalues <- c(pvalues, wilcox.test(thetayc~match, data=ctyc, paired=T)$p.value)

dtyc <- subset(tyc, match == 'LS_RS' | match == 'LB_RB')
dtyc <- dtyc[-25,]
pvalues <- c(pvalues, wilcox.test(thetayc~match, data=dtyc, paired=T)$p.value)

etyc <- subset(tyc, match == 'LS_RS' | match == 'LB_LS')
etyc <- etyc[-25,]
pvalues <- c(pvalues, wilcox.test(thetayc~match, data=etyc, paired=T)$p.value)

ftyc <- subset(tyc, match == 'LB_RB' | match == 'LB_LS')
pvalues <- c(pvalues, wilcox.test(thetayc~match, data=ftyc, paired=T)$p.value)

pvalues <- p.adjust(pvalues, method = "BH")

# now for exit comparisons

stoolpvalues <- c()

htyc <- subset(tyc, match=='RB_SS' | match=='RS_SS')
htyc <- htyc[-25,]
stoolpvalues <- c(stoolpvalues, wilcox.test(thetayc~match, data=htyc, paired=T)$p.value)

ityc <- subset(tyc, match=='RB_SS' | match=='LB_SS')
stoolpvalues <- c(stoolpvalues, wilcox.test(thetayc~match, data=ityc, paired=T)$p.value)

jtyc <- subset(tyc, match=='RB_SS' | match=='LS_SS')
stoolpvalues <- c(stoolpvalues, wilcox.test(thetayc~match, data=jtyc, paired=T)$p.value)

ktyc <- subset(tyc, match=='RS_SS' | match=='LB_SS')
ktyc <- ktyc[-25,]
stoolpvalues <- c(stoolpvalues, wilcox.test(thetayc~match, data=ktyc, paired=T)$p.value)

ltyc <- subset(tyc, match=='RS_SS' | match=='LS_SS')
ltyc <- ltyc[-25,]
stoolpvalues <- c(stoolpvalues, wilcox.test(thetayc~match, data=ltyc, paired=T)$p.value)

mtyc <- subset(tyc, match=='LB_SS' | match=='LS_SS')
stoolpvalues <- c(stoolpvalues, wilcox.test(thetayc~match, data=mtyc, paired=T)$p.value)

stoolpvalues <- p.adjust(stoolpvalues, method = "BH")

alltyc <- read.table("../data/process/allshared.summary", sep = '\t', header = T, row.names=NULL)
alltyc <- separate(alltyc, label, into= c('pt1', 'samp1'), sep="-", remove=F)
alltyc <- separate(alltyc, comparison, into= c('pt2', 'samp2'), sep="-", remove=F)
alltyc <- alltyc[-1]
alltyc <- alltyc[-7]

#ultimately want a plot of all points where pt1 == pt2 in one bar and all of the others in another column 
#unite and make column of 0/1 for matches? then can plot 1 and 0s 
#should i separate out lumen and mucosa ? sure or no not for now

alltyc["same_pt"] <- NA

for (i in 1:nrow(alltyc)){
  if (alltyc$pt1[i] == alltyc$pt2[i]){
    alltyc$same_pt[i] <- 1
  }
  else alltyc$same_pt[i] <- 0
}

inter_medians <- aggregate(thetayc ~ same_pt, alltyc, median)
```

```{r randomforest, echo=FALSE}

#make OTU abundance file
#Create df with relative abundances
shared_file_rf <- subset(shared_file, select = -c(numOtus, label))
shared_meta_rf <- merge(meta_file, shared_file, by.x='group', by.y='row.names')

#create dataframe with relative abundances
rel_abund <- 100*shared_file_rf/unique(apply(shared_file_rf, 1, sum))

#Create vector of OTUs with median abundances >1%
OTUs_1 <- apply(rel_abund, 2, max) > 1
OTU_list <- colnames(rel_abund)[OTUs_1]
#get df of just top OTUs
rel_abund_top <- rel_abund[, OTUs_1]
rel_meta <- merge(meta_file, rel_abund_top, by.x='group', by.y="row.names")

seed <- 1
n_trees <- 2001

source('../code/random_functions.R')
source('../code/tax_level.R')

#####RandomForest###########################################################################################
#build randomForest model for each location comparison using randomize_loc function 
rf_distal <- randomize_loc(rel_meta, "LB", "LS") #OOB 10.26%
rf_proximal <- randomize_loc(rel_meta, "RB", "RS") #OOB 53%
rf_bowel <- randomize_loc(rel_meta, "LB", "RB") #OOB 25.64%
rf_lumen <- randomize_loc(rel_meta, "LS", "RS") #OOB 69.23%

#and for each site
rf_all <- randomize_site(rel_meta, "mucosa", "stool")


#####AUCRF#####################################################################################

# create RF model with AUCRF outputs top OTUs
aucrf_data_distal_bs <- auc_loc(rel_meta, "LB", "LS")
aucrf_data_LRbowel <- auc_loc(rel_meta, "LB", "RB")
aucrf_data_proximal_bs <- auc_loc(rel_meta, "RB", "RS")
aucrf_data_LRlumen <- auc_loc(rel_meta, "LS", "RS")
aucrf_data_allum <- auc_site(rel_meta, "mucosa", "stool")

########Cross-Validation#################################################################
#10 fold cross validation for all lumen vs mucosa 
iters <- 100
cv10f_aucs <- c()
cv10f_all_resp <- c()
cv10f_all_pred <- c()
for(j in 1:iters){
  set.seed(j)
  sampling <- sample(1:nrow(aucrf_data_allum),nrow(aucrf_data_allum),replace=F)
  cv10f_probs <- rep(NA,78)
  for(i in seq(1,77,7)){
    train <- aucrf_data_allum[sampling[-(i:(i+6))],]
    test <- aucrf_data_allum[sampling[i:(i+6)],]
    set.seed(seed)
    temp_model <- AUCRF(site~., data=train, pdel=0.99, ntree=500)
    cv10f_probs[sampling[i:(i+6)]] <- predict(temp_model$RFopt, test, type='prob')[,2]
  }
  cv10f_roc <- roc(aucrf_data_allum$site~cv10f_probs)
  cv10f_all_pred <- c(cv10f_all_pred, cv10f_probs)
  cv10f_all_resp <- c(cv10f_all_resp, aucrf_data_allum$site)
  cv10f_aucs[j] <- cv10f_roc$auc #stores aucs for all iterations, can use to calc IQR
}
cv10f_roc <- roc(cv10f_all_resp~cv10f_all_pred)

#10fold CV for L lumen vs L mucosa
iters <- 100
cv10f_aucs <- c()
cv10f_all_resp_distal_bs <- c()
cv10f_all_pred_distal_bs <- c()
for(j in 1:iters){
  set.seed(j)
  sampling <- sample(1:nrow(aucrf_data_distal_bs),nrow(aucrf_data_distal_bs),replace=F)
  cv10f_probs <- rep(NA,39)
  for(i in seq(1,36,4)){
    train_distal_bs <- aucrf_data_distal_bs[sampling[-(i:(i+3))],]
    test_distal_bs <- aucrf_data_distal_bs[sampling[i:(i+3)],]
    set.seed(seed)
    temp_model_distal_bs <- AUCRF(location~., data=train_distal_bs, pdel=0.99, ntree=500)
    cv10f_probs[sampling[i:(i+3)]] <- predict(temp_model_distal_bs$RFopt, test_distal_bs, type='prob')[,2]
  }
  cv10f_roc_distal_bs <- roc(aucrf_data_distal_bs$location~cv10f_probs)
  cv10f_all_pred_distal_bs <- c(cv10f_all_pred_distal_bs, cv10f_probs)
  cv10f_all_resp_distal_bs <- c(cv10f_all_resp_distal_bs, aucrf_data_distal_bs$location)
  cv10f_aucs[j] <- cv10f_roc_distal_bs$auc #stores aucs for all iterations, can use to calc IQR
}
cv10f_roc_distal_bs <- roc(cv10f_all_resp_distal_bs~cv10f_all_pred_distal_bs)

#10fold CV for R lumen vs R mucosa
iters <- 100
cv10f_aucs <- c()
cv10f_all_resp_proximal_bs <- c()
cv10f_all_pred_proximal_bs <- c()
for(j in 1:iters){
  set.seed(j)
  sampling <- sample(1:nrow(aucrf_data_proximal_bs),nrow(aucrf_data_proximal_bs),replace=F)
  cv10f_probs <- rep(NA,39)
  for(i in seq(1,36,4)){
    train_proximal_bs <- aucrf_data_proximal_bs[sampling[-(i:(i+3))],]
    test_proximal_bs <- aucrf_data_proximal_bs[sampling[i:(i+3)],]
    set.seed(seed)
    temp_model_proximal_bs <- AUCRF(location~., data=train_proximal_bs, pdel=0.99, ntree=500)
    cv10f_probs[sampling[i:(i+3)]] <- predict(temp_model_proximal_bs$RFopt, test_proximal_bs, type='prob')[,2]
  }
  cv10f_roc_proximal_bs <- roc(aucrf_data_proximal_bs$location~cv10f_probs)
  cv10f_all_pred_proximal_bs <- c(cv10f_all_pred_proximal_bs, cv10f_probs)
  cv10f_all_resp_proximal_bs <- c(cv10f_all_resp_proximal_bs, aucrf_data_proximal_bs$location)
  cv10f_aucs[j] <- cv10f_roc_proximal_bs$auc #stores aucs for all iterations, can use to calc IQR
}
cv10f_roc_proximal_bs <- roc(cv10f_all_resp_proximal_bs~cv10f_all_pred_proximal_bs)

#10 fold cross validation for L vs R mucosa

iters <- 100
cv10f_aucs_muc <- c()
cv10f_all_resp_muc <- c()
cv10f_all_pred_muc <- c()
for(j in 1:iters){
  set.seed(j)
  sampling_muc <- sample(1:nrow(aucrf_data_LRbowel),nrow(aucrf_data_LRbowel),replace=F)
  cv10f_probs_muc <- rep(NA,39)
  for(i in seq(1,36,4)){
    train_muc <- aucrf_data_LRbowel[sampling_muc[-(i:(i+3))],]
    test_muc <- aucrf_data_LRbowel[sampling_muc[i:(i+3)],]
    set.seed(seed)
    temp_model_muc <- AUCRF(location~., data=train_muc, pdel=0.99, ntree=500)
    cv10f_probs_muc[sampling_muc[i:(i+3)]] <- predict(temp_model_muc$RFopt, test_muc, type='prob')[,2]
  }
  cv10f_roc_muc <- roc(aucrf_data_LRbowel$location~cv10f_probs_muc)
  cv10f_all_pred_muc <- c(cv10f_all_pred_muc, cv10f_probs_muc)
  cv10f_all_resp_muc <- c(cv10f_all_resp_muc, aucrf_data_LRbowel$location)
  cv10f_aucs_muc[j] <- cv10f_roc_muc$auc #stores aucs for all iterations, can use to calc IQR
}
cv10f_roc_muc <- roc(cv10f_all_resp_muc~cv10f_all_pred_muc)


#10 fold cross validation for L vs R lumen
iters <- 100
cv10f_aucs_lum <- c()
cv10f_all_resp_lum <- c()
cv10f_all_pred_lum <- c()
for(j in 1:iters){
  set.seed(j)
  sampling_lum <- sample(1:nrow(aucrf_data_LRlumen),nrow(aucrf_data_LRlumen),replace=F)
  cv10f_probs_lum <- rep(NA,39)
  for(i in seq(1,36,4)){
    train_lum <- aucrf_data_LRlumen[sampling_lum[-(i:(i+3))],]
    test_lum <- aucrf_data_LRlumen[sampling_lum[i:(i+3)],]
    set.seed(seed)
    temp_model_lum <- AUCRF(location~., data=train_lum, pdel=0.99, ntree=500)
    cv10f_probs_lum[sampling_lum[i:(i+3)]] <- predict(temp_model_lum$RFopt, test_lum, type='prob')[,2]
  }
  cv10f_roc_lum <- roc(aucrf_data_LRlumen$location~cv10f_probs_lum)
  cv10f_all_pred_lum <- c(cv10f_all_pred_lum, cv10f_probs_lum)
  cv10f_all_resp_lum <- c(cv10f_all_resp_lum, aucrf_data_LRlumen$location)
  cv10f_aucs_lum[j] <- cv10f_roc_lum$auc #stores aucs for all iterations, can use to calc IQR
}
cv10f_roc_lum <- roc(cv10f_all_resp_lum~cv10f_all_pred_lum)
```
```{r important_otus, echo=FALSE}
#just LB vs LS 
distal_otu_feat <- colnames(aucrf_data_distal_bs[2:6])
otu_taxa_distal <- get_tax(1, distal_otu_feat, tax_file)
#Abundance stripchart or most predictive otus
ls_abunds <- shared_meta[shared_meta$location=='LS', distal_otu_feat]/10000 + 1e-4
lb_abunds <- shared_meta[shared_meta$location=='LB', distal_otu_feat]/10000 + 1e-4

proximal_otu_feat <- colnames(aucrf_data_proximal_bs[2:6])
otu_taxa_proximal <- get_tax(1, proximal_otu_feat, tax_file)
#Abundance stripchart or most predictive otus
rs_abunds <- shared_meta[shared_meta$location=='RS', proximal_otu_feat]/10000 + 1e-4
rb_abunds <- shared_meta[shared_meta$location=='RB', proximal_otu_feat]/10000 + 1e-4

#Lb vs Rb
LRbowel_otu_feat <- colnames(aucrf_data_LRbowel[2:6])
otu_taxa_LRbowel <- get_tax(1, LRbowel_otu_feat, tax_file)
#Abundance stripchart or most predictive otus 
lb_abunds <- shared_meta[shared_meta$location=='LB', LRbowel_otu_feat]/10000 + 1e-4
rblb_abunds <- shared_meta[shared_meta$location=='RB', LRbowel_otu_feat]/10000 + 1e-4

#LS vs RS
LRlumen_otu_feat <- colnames(aucrf_data_LRlumen[2:6])
otu_taxa_LRlumen <- get_tax(1, LRlumen_otu_feat, tax_file)
#Abundance stripchart or most predictive otus 
lsrs_abunds <- shared_meta[shared_meta$location=='LS', LRlumen_otu_feat]/10000 + 1e-4
rsls_abunds <- shared_meta[shared_meta$location=='RS', LRlumen_otu_feat]/10000 + 1e-4

```


```{r oncogenic_otus, echo=FALSE}
subsampled_meta <- merge(meta_file, subsample, by.x='group', by.y='row.names')
fuso179 <- subsampled_meta[, colnames(subsampled_meta) %in% c("group", "patient", "location", "Otu00179")]
subsampled_to = 4231

fuso179[,5] <- (fuso179[,4]/subsampled_to)*100
names(fuso179)[5] <- "Otu00179_relAbund"

p_152 <-subsampled_meta[, colnames(subsampled_meta) %in% c("Group", "patient", "location", "Otu00152")]

p_152[,5] <- (p_152[,4]/subsampled_to) *100
names(p_152)[5] <- "Otu152_abund"

fusoviv <- subsampled_meta[, colnames(subsampled_meta) %in% c("group", "patient", "location", "Otu00472")]
fusoviv[,5] <- (fusoviv[,4]/4321)*100
names(fusoviv)[5] <- "Otu00472_relAbund"
```


### Abstract

**Background and Aims:**
The microbiome has been implicated in the development of colorectal cancer (CRC). CRC tumors of the proximal and distal colon are morphologically and genetically distinct but the microbes that reside in each location are not well described. Prior studies have used fecal samples from healthy and CRC patients to examine disease-associated microbes, which cannot reflect different biogeographic sites in the colon. Here we profile the microbes of the healthy proximal and distal mucosa and lumen.  

**Methods:**
We used a two-colonoscope sampling approach on 20 healthy subjects that had not undergone any bowel preparation procedure. 16S rRNA gene sequencing was performed on proximal and distal mucosal and luminal biopsies and home-collected stool. Alpha and beta diversity metrics for each site were calculated. Random Forest classification models were built using taxa abundance and sample location.

**Results:**
Beta diversity analysis showed that the right mucosa and lumen were the most similar to each other. The distal mucosa had higher abundance of _Finegoldia, Murdochiella_, _Peptoniphilus_, _Porphyromonas_ and  _Anaerococcus_. The proximal mucosa had more of the genera _Enterobacteriaceae_, _Bacteroides_ and _Psuedomonas_. The classification model performed well when classifying mucosal samples into proximal or distal sides (AUC = 0.8500). Separating proximal and distal luminal samples proved more challenging (AUC = 0.5803) and specific microbiota that differentiated the two were hard to identify. 

**Conclusions:**
By sampling the unprepped human colon, we identified distinct bacterial populations native to the proximal and distal sides. Further investigation of these bacteria may elucidate if and how these groups contribute to different oncogenic processes on the respective sides of the colon. 

Words: 260/260

Keywords: microbiome; colon cancer; proximal and distal colon

### Introduction

The human colon is an ecosystem comprised of several different microenvironments inhabited by resident bacterial members of the microbiome. Concentrations of oxygen, water and anti-microbial peptides change along the gut axis and influence what populations of microbes reside in each location. Microenvironments differ not only longitudinally along the colon, but latitudinally from the epithelium to mucosa to intestinal lumen, offering several sites for different microbial communities to flourish. The identity of these specific microbes and communities are important for understanding the etiology of complex colon diseases such as Inflammatory Bowel Disease (IBD) and Colorectal Cancer (CRC). IBD and CRC are known to be preceded or accelerated by perturbations in gut microbes [@Yamauchi2012; @Forbes2016; @Halfvarson2017]. The manifestations of these diseases are known to vary based upon the biogeographical location in which they occur. For instance, CRC that arise in the distal (left) colon are of hindgut origin and tend to have large chromosomal alterations indicative of chromosomal instability @Yamauchi2012. In contrast, CRCs arising in the proximal (right) colon tumors are of midgut origin and tend to be sessile and microsatellite instable (MSI with BRAF and KRAS mutations) @Yamauchi2012. The distal and proximal sides of the colon differ in the amount of inflammation present and the genomic instability of precancerous cells, respectively, in addition to variation in the previously mentioned chemical gradients [@Yamauchi2012; @Albenberg2014; @Donaldson2015]. In IBD patients, disease occurring in the distal colon extending proximally is usually indicative of ulcerative colitis (UC), whereas Crohn's disease (CD) can occur anywhere along the GI tract, most commonly in the ileum and the cecum @Forbes2016. UC presents as continuous disease with only mucosal involvement, where as CD has skip lesions and full thickness involvement that may cause abscesses, strictures and fistulas @Forbes2016. Thus, given the varied physiology of the proximal-distal axis of the colon and known differences in disease patterns at these sites, symbiotic microbes and their metabolites likely vary as well, and may influence the heterogenous disease prognoses of IBD and CRC. Because CRC can be a long-term complication of IBD, the distribution of microbes is important to understanding the pathophysiology of both diseases. 

Several recent findings have shown that development and progression of IBD or CRC can be attributed to specific molecular events as a result of interactions between the gut microbiota and human host [@Yamauchi2012; @Halfvarson2017]. For instance, comparison of the bacteria present on CRC tumors with those found on nearby healthy tissue has identified specific species that are tumor-associated @Dejea2014. These species include the oral pathogens _Fusobacterium nucleatum_ and _Porphyromonas asacharolytica_. Interestingly, these periodontal pathogens have been highly predictive of whether a patient had CRC tumors or not in our prior human stool classification studies @Baxter2016. _F. nucleatum_ has also been found to be elevated in the stool and biopsies of patients with IBD as compared to healthy controls @Strauss2011. Furthermore, studies of _F. nucleatum_ isolated from mucosal biopsies showed that more invasive  _F. nucleatum_ positively correlates with IBD disease level @Strauss2011. Like many intestinal pathogens, the bacteria appear to have a high-impact despite being lowly-abundant in the community @Forbes2016. The physiology of these rare taxa may contribute to the colonic disease state. These studies often examined only shed human stool or the small intestine, preventing fine-resolution analysis of paired samples from the proximal and distal sides of the colon. Similarly, comparisons of on- or off-tumor/lesion bacteria rarely have matched tissue from the other side of the colon from the same patient, limiting what conclusions can be drawn about the colonic microbiome overall, let alone at that specific site. Due to these limitations, the contribution of the gut microbiota to IBD and CRC disease location in the colon is largely undefined. Characterizing these communities could provide needed insight into disease etiology, including how the disruption of the healthy community could promote the initiation or proliferation of the distinct proximal and distal CRC tumors or IBD flares.

The few existing profiles of the microbial biogeography of the gut have been limited by sample collection methods. The majority of human gut microbiome studies have been performed on whole shed feces or on samples collected during colonoscopy or surgery. While the latter method allows investigators to acquire samples from inside the human colon, typically these procedures are preceded by the use of bowel preparation methods such as the consumption of laxatives to cleanse the bowel. Bowel preparation is essential for detecting cancerous or precancerous lesions in the colon, but complicates microbiome profiling as the chemicals strip the bowel of contents and disrupt the mucosal layer [@Jalanka2014; @Harrell2012]. As such, what little information we do have about the biogeographical distribution of the microbes in the proximal and distal colon is confounded by the bowel preparation procedure. 

Here we address the limitations of previous studies and effectively identify the microbes specific to the lumen and mucosa of the proximal and distal healthy human colon. We used an unprepared colonoscopy technique to sample the natural community of each location of the gut without prior disruption of the native bacteria in 20 healthy volunteers. To address the inherent inter-individual variation in human microbiomes, we used a machine-learning classification algorithm trained on curated 16S rRNA sequencing reads to identify microbes specific to each location. We found that our classification models were able to separate mucosal and lumenal samples as well as differentiate between sides of the colon based on populations of specific microbes. By identifying the specific microbes we are poised to ask if and how the presence or disruption of the microbes at each site contribute to the development of the specific tumor subtypes of CRC in the proximal and distal human colon. 


### Results

#### Microbial membership and diversity of the proximal and distal colon 
Lumenal and mucosal samples were collected from the proximal and distal colon of 20 healthy humans that had not undergone bowel preparation (Figure 1). Participants also collected stool at home one week prior to the procedure. To characterize the bacterial communities present at these sites, 16S rRNA gene sequencing was performed on extracted DNA from each sample. As expected, each site was primarily dominated by _Firmicutes_ and _Bacteriodetes_ (Figure 2A) @LloydPrice2016. Samples had varying levels of diversity at each site, irrespective of the individual (Figure 2B). For example, the proximal mucosa was more diverse than the distal for some individuals while the opposite was true for others. Therefore we could not identify a clear pattern of changes in microbial diversity along the gut axis. 

To compare similarity between sides (proximal or distal) or sites (lumen or mucosa), we calculated distances from Operational Taxonomic Unit (OTU) abundances and compared these distances for all individuals. Again, across all patient samples we observed a range of distances when comparing sample locations (Figure 3A) and again those ranges did not follow a clear pattern on an individual basis. However, when comparing median distances between the proximal lumen and mucosa, the proximal versus distal lumen, the proximal versus distal mucosa, and the distal lumen and mucosa, we found that the proximal lumen and mucosa were most similar to each other than the other samples (P < 0.005, Wilcoxon, BH adjustment). 

#### Fecal samples resemble lumenal samples from the distal colon 
Next, we calculated distances to examine how each sample compared to the home-collected feces. Amidst variability between patients, we did identify significantly smaller distance between the distal lumenal sample and the feces (Figure 3B, P < 0.05, Wilcoxon, BH adjustment). Furthermore, there was an even larger difference in the comparisons of the distal mucosa to the feces, indicating that the mucosa is different from the stool as compared to lumen (P < 0.0005, Wilcoxon, BH adjustment). To determine what factors may be driving the differences seen among the samples, we compared distances between samples from all subjects (interpersonal) versus samples from within one subject (intrapersonal). We found that samples from one individual were far more similar to each other than to other study subjects (Figure 3C), consistent with previous human microbiome studies that have sampled multiple sites of the human colon [@Eckburg2005; @deCarcer2010; @Zhang2013]. Thus interpersonal variation between subjects drove the differences between samples more than sample site or location. Overall, the results comparing the structure of the communities suggest that the contents of the distal lumen are most representative of the patient's feces, and the microbes remaining on the mucosa are more distinct. 

#### Random Forest classification models identify important Operational Taxonomic Units (OTUs) on each side
To identify OTUs that were distinct at each biogeographical site, we constructed several Random Forest models trained using OTU abundances. We used 10-fold cross validation to build the first model to classify the lumenal versus mucosal samples for the proximal and distal sides, independently (Figure S1A). The models performed well when classifying these samples (proximal AUC = 0.764, distal AUC = 0.908). The OTUs that were most predictive of each site were identified by their greatest mean decrease in accuracy when removed from the model. For distinguishing the proximal lumen and mucosa, OTUs from the _Bacteriodes_, _Actinomyces_, _Psuedomonas_ and two OTUs from the _Enterobacteraceae_ genera were differentially abundant (Figure 4A). The model classifying the distal lumen and mucosa identified OTUs from _Turicibacter_, _Finegoldia_, _Peptoniphilus_ and two OTUs from the _Anaerococcus_ genera (Figure 4B). These results indicated that there were fine differences between the different sites of the colon, and that these could be traced to specific OTUs on each side. 

Next, we built a model to differentiate the proximal and distal lumenal samples using 10-fold cross validation. The model performed best when distinguishing the proximal versus distal mucosa (Figure S1B, AUC = 0.850) compared to the proximal versus distal lumen (AUC = 0.580). OTUs that were differentially abundant between the distal and proximal mucosa included members of the _Porphyromonas_, _Murdochiella_, _Finegoldia_, _Anaerococcus_ and _Peptoniphilus_ genera (Figure 5A). Differentially abundant OTUs of the proximal and distal lumen included three OTUs of the _Bacteroides_ genus, a _Clostridium IV_ OTU and an _Oscillibacter_ OTU (Figure 5B). This analysis found that some of the same OTUs that are distinct between the mucosa and lumen also helped to differentiate between the two sides- such as _Anaerococcus_ and _Finegoldia_. 

#### Bacterial OTUs associated with cancer are found in healthy individuals 
Given that specific bacterial species have been associated with colorectal cancer and IBD, we probed our sample set for these OTUs. Among our 100 samples, the most frequent sequence associated with the _Fusobacterium_ genus was OTU179, which aligns via BLASTn to _Fusobacterium nucleatum subsp animalis_ (100% over full length). This is the only species of _Fusobacterium_ known to have oncogenic properties and be found on the surfaces of colorectal cancer tumors. @Castellarin2011. There were 14 patient samples with the _F. nucleatum subsp. animalis_ sequences. Of the samples with the highest abundance of _F. nucleatum subsp. animalis_, four of the samples were from the proximal mucosa and three from the distal mucosa (Figure S2A). The second most frequent _Fusobacterium_ sequence was OTU472, which aligned with 99% identity to _F. varium_. In addition to _F. nucleatum_, _F. varium_ has been associated with IBD @Lee2016. Four study participants harbored _F. varium_ and the samples were split evenly between the proximal and distal mucosa (Figure S2B). OTU152 was similar to the members of the _Porphyromonas_ genus and the most frequent sequence in that OTU aligned to _Porphyromonas asacharolytica_ (99% over full length), another bacterium commonly detected and isolated from colorectal tumors. OTU152 was only detected on the distal mucosa, and in fact was one of the OTUs the classification model identified as separating distal and proximal sides (Figure S2C). Among the 11 distal mucosa samples that were positive for _P. asacharolytica_, the relative abundances for this OTU ranged from 0.01% - 16%. Thus, disease-associated OTUs could be found in our sample set of 20 healthy individuals. 

### Discussion
Here we identified bacterial taxa that were specific to the lumen and mucosa of the proximal and distal sides of the human colon from samples collected during unprepared colonoscopy. We found that all locations contained a range of phyla abundances and a range of diversity, but that there was a wide variability between subjects. Pairwise comparisons of each of the sites revealed that the proximal mucosa and lumen were most similar to each other. Further, comparison of colonoscopy-collected samples with samples collected from stool at home showed that the distal lumen was most similar to feces. Random Forest models built on OTU relative abundances from each sample identified microbes that were particular to each location of the colon. Finally, we were able to detect some bacterial OTUs associated with colonic disease in our healthy patient cohort. Using unprepped colonoscopy and machine learning, we have identified bacterial taxa specific to the healthy proximal and distal human colon. 

When examining the relative abundance of the dominant phyla at each site (i.e. _Bacteriodes_ and _Firmicutes_), there was a wide amount of variation. This likely reflects not only the variability between human subjects, casued by differences in age, sex, diet, but also biogeogrphical "patchiness" in the gut microbiome. Several studies have noted that the bacteria recoverable from the same mucosal sample location can be vastly different when the samples are taken just 1 cm away from each other @Hong2011. Similar patchiness is also observed in lumenal contents and fecal samples themselves; there is observed separation of different interacting microbes along the length of a stool sample, for instance @Stearns2011. That said, across our samples the mucosal samples harbor more _Proteobacteria_, consistent with previous studies comparing mucosal swabs to lumenal content in humans @Albenberg2014. Hence, the conclusions we can draw from phyla analysis may be impacted by patchiness between subjects. 

To get around the noisiness from a diverse set of samples, we built a Random Forest model to identify microbes specific to each side. For each comparison we identified the top five OTUs that were strongly predictive of one site or another. Generally, OTUs identified in each location were consistent with known physiological gradients along the gut axis @Donaldson2015. For instance, the proximal mucosa contains the highest oxygen concentratons of the colon and  harbored mucosa-associated facultative anaerobes such as _Actinomyces_ and _Enterobacteraceae_ and aerobic _Psuedomonas_. The distal mucosa was far more likely to host strictly anaerobic species such as _Porphyromonas_, _Anaerococcus_, _Finegoldia_ and _Peptoniphilus_. Thus the gut microenvironment of each location likely enriches for these specific microbes. 

In addition to identifying features that are specific to each side of the gut, the ability of the Random Forest to classify samples can serve as a proxy for similarity. That is, a higher AUC value indicates the samples are more efficiently classfied (and thus more different) than a model with a lower AUC value. For instance, the model separating the proximal and distal mucosa has an AUC of 0.850 whereas the model for classifying the proximal and distal lumen has a much lower AUC of 0.580. Further, the latter model required 44 OTUs to best separate the samples whereas the models separating the mucosa only needed 10 OTUs. The much lower AUC and need for a high number of features compared to other models suggest these locations are the most similar of the comparisons tested. We speculate that the model was least effective at classifying the proximal and distal lumenal contents because the samples are arguably composed of the same bacteria but differ in water content. 

We detected _F. nucleatum_ and _P. asacharolytica_ in 8 and 5 of our subjects, respectively. These bacteria have been shown to be predictive of colorectal cancer in humans @Baxter2016 and have oncogenic properties in cell culture and in mice @Sears2014. Interestingly, while _F. nucleatum_ was found on both sides of the colon, _P. asacharolytica_ was only detected in the distal mucosa. Not much is known about the distribution of _P. asacharolytica_ but given its documented anaerobic characteristics and asacharolytic metabolism, it might not be surprising that it resides in the less-oygen-rich and proteinaceous distal mucosa @Albenberg2014. In studies examining bacteria on colorectal cancer tumors, _F. nucleatum_ is more commonly detected on proximal-sided tumors, and distribution of _F. nucleatum_ decreases along the colon to rectum @Mima2016.  Of the 8 (40%) individuals positive for _F. nucleatum_ in this study, the bacterium was spread across the proximal mucosa, distal lumen and distal mucosa. Data examining bacterial biofilms on the mucosa of CRC tumors suggests that _Fusobacteria_ species are more commonly found on proximal tumors and in biofilms, indicating that it is not only the presence of the bacteria but the organization of the tumor community that contributes to _Fusobacterium's_ role in tumorigenesis @Dejea2014. Finally, _Fusobacterium_ and _Porphyromonas_ species have been known to not only co-occur on CRC tumors but also to synergistically promote tumorigenesis in an oral cancer model @Whitmore2014 @Flynn2016. Thus, further analysis of the distribution and activities of these pathogens may elucidate a mechanism for development of IBD or CRC subtypes in the proximal or distal colon. 

The _Fusobacterium_ species _nucleatum_ and _varium_ have been commonly isolated from mucosal biopsies of patients with IBD @Lee2016. Laboratory experiments with these isolates have shown that disease-isolated _F. nucleatum_ are more invasive and stimulate more TNF-$\alpha$ production than strains from healthy individuals @Strauss2011, suggesting the bacteria may increase inflammation in the gut as well @Dharmani2011. _F. varium_ isolated from UC patients caused colonic ulcers in an experimental mouse model @Ohkusa2003. _F. varium_ was only detected in three of our study participants and two of those samples were isolated form the proximal mucosa (Figure S2B).  _F. varium_ is most commonly isolated from UC patient biopsies from the ileum or cecum @Ohkusa2002, suggesting this species may exhibit preference for the different environmental conditions of these gastrointestinal sites. Further work will assess how gut environment may select for species which may then cause localized disease. 

Specific comparisons of our findings to previously published gut biogeography studies are confounded by the use of bowel preparation methods in most other studies. A rare report of a matched-colonoscopy study sampled 18 patient's colonic mucosa and lumenal contents prior to and after bowel cleansing @Shobar2016. This study found that mucosal and lumenal samples were distinguishable prior to bowel cleansing, but that bowel preparation resulted in an increase in shared OTUs between each site @Shobar2016. After seven days, bowel cleansing not only made the samples harder to distinguish, but it also decreased in diversity across sites. Bowel preparation clearly biases into the microbes recovered from sampling the lumen or mucosa of a prepared bowel.  

By revealing specific differences in microbial populations at each location in the gut via sampling an unprepared bowel, we can begin to form hypothesies about how specific host-microbe interactions can affect disease progression of proximal and distal CRC and IBD subtypes. To this point, 16S rRNA gene sequecing community profiling studies do not provide enough information to fully probe these questions. In particular, 16S sequencing cannot not profile the host characteristics at each site. Combining the unprepared colonoscopy approach with analysis of multi-omic sequencing data may be useful in further characterizing host-microbiome interactions along the gut axis for both health and disease. 

### Acknowledgments 
We thank all the individuals who volunteered for the study. This work was supported by the Rose and Lawrence C. Page Foundation (DKT). We would also like to thank Brian Kleiner, Chelsea Crofoot, and Kirk Herman for their roles in study coordination, subject recruitment, sample collection and sample processing. <need grant # here>

### Methods
#### Human subjects
The procedures in this study and consent were approved by the Institutional Review Board at the University of Michigan Health System with protocol number XXXX. Subjects were recruited using the online recruitment platform and were pre-screened prior to enrollment in the study. Exclusion criteria included: use of asprin or NSAIDs within 7 days, use of antibiotics within 3 months, current use of anticoagulants, known allergies to Fentanyl or Benadryl, prior history of colon disease, diabetes, abdominal surgery, respiratory, liver, kidney or brain impairments, undergoing current chemotherapy or radiation treatment and subjects that were pregnant or trying to conceive. 20 subjects that met the criteria were selected and provided signed informed consent prior to the procedure. There were 13 female and 7 male subjects ranging in age from 25 to 64. 

#### Sample collection
At a baseline visit, subjects were consented and given a home collection stool kit (Source of kit?). At least one week prior to the scheduled colonoscopy, subjects were to collect whole stool at home and ship the samples to a research coordinator on ice. Notably, subjects did not undergo any bowel preparation method prior to sampling. On the procedure day, subjects reported to the Michigan Clinical Research Unit at the University of Michigan Health System. Patients were consciously sedated using Fentanyl, Versed and/or Benadryl as appropriate. A flexible sigmoidoscope was first inserted about 25cm into the colon and endoscopy brush used to collect lumenal/stool contents. Two lumenal samples were collected and the contents immediately deposited into RNAlater (Fischer) and flash-frozen in liquid nitrogen. The brushes were withdrawn and biopsy forceps were used to collect mucosal biopsies on sections of the colon that were pink and free of stool matter. Three mucosal biopsies were collected and flash-frozen in RNAlater. These samples comprised the distal or distal colon samples. The sigmoidoscope was then withdrawn and a pediatric colonoscope was inserted to reach the ascending colon. Samples were then collected as in the distal colon and the colonoscope withdrawn. All samples were stored at -80$^\circ$C.

#### Sample processing, sequencing and analysis
DNA extraction was performed using the PowerMicrobiome DNA/RNA Isolation Kit (MO BIO Laboratories). For tissue biopsies, Bond-Breaker TCEP solution (Fisher) and 2.8mm ceramic beads (MO BIO Laboratories) were added to the bead beating step to enhance DNA recovery from mucosal samples. The resulting DNA was used as template for amplification of the V4 region of the 16S rRNA gene and fragments were sequenced on an Illumina MiSeq as previously described @Kozich2013. Sequences were curated using the mothur software as described previously @Schloss2009. The sequences were assigned taxonomic classification using a naive Bayesian classifier trained using a 16S rRNA gene training set from the Ribosomal Database Project (RDP) @Wang2007 and clustered into operational taxonomic units (OTUs) based on a 97% similarity cutoff. Sequencing and analysis of a mock community revealed the error rate to be 0.018%. Samples were rarefied to 4231 sequences per sample in order to reduce uneven sampling bias. 

Diversity analysis was performed using the Simpson diversity calculator and $\theta$<sub>YC</sub> calculator metrics in mothur version 1.39.5 @Schloss2009. ThetaYC distances were calculated to determine the dissimilarity between two samples. Random Forest classification models were built using the randomForest R package and resultant models were used to identify the OTUs that were most important for classifying each location @Liaw2002. To get species-level information about sequences of interest, sequences were aligned using blastn and the species name was only used if the identity score was $\ge$ 99% over the full-length of the contig and matched a single reference. 

#### Statistical analysis
Differences in community membership at the phyla level were tested using the analysis of molecular variance (AMOVA) metric in mothur. Differences in $\theta$<sub>YC</sub> distances by location were tested using the Wilcoxon rank-sum test adjusted for multiple comparisons using the Benjamini-Hochberg procedure.

#### Data availability
16S rRNA gene sequence reads and experiment metadata are available on the NCBI Sequence Read Archive (SRA) with accession number XXXX. A reproducible data analysis pipeline can be found at https://github.com/SchlossLab/Flynn_LRColon_XXXX_2017.

\newpage

### Figures

####Figure 1


Sampling strategy. A flexible sigmoidoscope was used to sample the distal colonic luminal contents and mucosa. The scope was inserted ~ 25cm into the subject and endoscopy brushes were used to sample the luminal contents (green star). A separate set of biopsy forceps was used to sample the distal mucosa (blue star). The sigmoidoscope was removed. A pediatric colonoscope was inserted and used to access the proximal colon. Biopsies were taken of the proximal luminal contents and mucosa as described. One week prior to the procedure stool was collected at home and sent into the laboratory. Representative images from one individual are shown.  

\newpage

####Figure 2
Microbial membership and diversity of the proximal and distal human colon. A) Relative abundance of the top five bacterial phyla in each sampling site. Each box represents the median and confidence intervals. B) Simpson diversity of the microbial communities at each location. The lines represent the median values. 

\newpage

####Figure 3
Distances of microbial community structure between sites of the gut. ThetaYC distances are shown for interpersonal similarities between two sites – each point represents one individual. In (A), comparisons of the proximal and distal mucosal and lumen are shown. In (B), comparisons of each site to the exit stool are shown. 

\newpage

####Figure 4
Taxa specific to the distal and proximal sides of the colon. Top five OTUs that are most important for the classification model for the distal mucosa and lumen (A) and the proximal mucosa and lumen (B). 

\newpage

####Figure 5 
Taxa specific to the distal and proximal mucosa and lumen. Top five OTUs that are most important for the classification model for the distal and proximal mucosa (A) and the distal and proximal lumen (B).

\newpage

####Figure S1
Random Forest classifies locations in the colon. A) Receiver Operator Characteristic curves are shown for the 10-fold cross validation of the Random Forest model classifying distal lumen versus proximal lumen (orange) and distal mucosa vs proximal mucosa (green). (B) Receiver Operator Characteristic curves are shown for the 10-fold cross validation of the Random Forest model classifying lumen and mucosal samples for the distal (red) and proximal (blue) sides of the colon. 

\newpage

####Figure S2
Location and abundance of cancer-associated OTUs. Relative abundance was calculated and plotted by sample site for each OTU of interest: (A) _Fusobacterium nucleatum subsp. animalis_ (B) _Fusobacterium varium_ and (C) _Porphyromonas asacharolytica_ 



### References 













