---
title: '**Machine learning identifies unique taxa differentiating proximal and distal
  human colonic microbiota**'
fontsize: 11pt
geometry: margin=1.0in
output:
  html_document: default
  pdf_document:
    includes: null
    keep_tex: yes
csl: plos.csl
---
\vspace{35mm}

Running title: Specific taxa differentiate proximal and distal human colon microbiota

\vspace{35mm}


Kaitlin J. Flynn^1^, Charles C. Koumpouras^1^, Mack T. Ruffin IV^2^, Danielle Kimberly Turgeon^3^, and Patrick D. Schloss^1$\dagger$^

\vspace{35mm}

$\dagger$ Corresponding author: pschloss@umich.edu


1\. Department of Microbiology and Immunology, University of Michigan Medical School, Ann Arbor, Michigan 48109

2\. Pennslyvania State University, Hershey, Pennslyvania ??

3\. Department of Internal Medicine, Division of Gastroenterology, University of Michigan Medical School, Ann Arbor, Michigan


\newpage
\linenumbers


### Abstract

Colorectal cancer (CRC) remains a leading cause of death worldwide. Tumors of the proximal (right) and distal (left) colon are morpologically and genetically distinct. Previous work from our group found that microbial dysbiosis is associated with the development of colorectal cancer tumors in studies of both mice and humans. Analysis of the fecal microbiota from healthy and CRC patients further revealed different microbial signatures associated with disease. In this study, we extended our observations of the fecal microbiome to analysis of the proximal and distal human colon. We used a two-colonoscope approach on subjects that had not undergone standard bowel preparation procedure. This technique allowed us to characterize the native proximal and distal luminal and mucosal microbiome without prior chemical disruption. 16S rRNA gene sequencing was performed on proximal and distal mucosal biopsies, luminal and exit stool for 20 healthy individuals. Diversity analysis of each location revealed that each site contained a diverse community, and that a patientâ€™s samples were more similar to each other than to that of other individuals. Since we could not differentiate sites along the colon based on community structure or community membership alone, we employed the machine-learning algorithm Random Forest to identify key species that distinguish biogeographical sites. Random Forest classification models were built using taxa abundance and sample location revealed distinct populations that were found in each location. Peptoniphilus, Anaerococcus, Enterobacteraceae, Pseudomonas and Actinomyces were most likely to be found in mucosal samples versus luminal samples (AUC = 0.925). The classification model performed well (AUC = 0.912) when classifying mucosal samples into proximal or distal sides, but separating luminal samples from each side proved more challenging (AUC = 0.755). The left mucosa was found to have high populations of Finegoldia, Murdochiella and Porphyromonas. Proximal and distal luminal samples were comprised of many of the same taxa, likely reflecting the fact that stool moves along the colon from the proximal to distal end. Finally, comparison of all samples to fecal samples taken at exit uncovered that the feces were most similar to samples taken from the left lumen, again reflecting the anatomical structure of the colon. Taken together, our results have identified distinct bacterial populations distinct of the proximal and distal colon. Further investigation of these bacteria may elucidate if and how these groups contribute to differential oncogenesis processes on the respective sides of the colon. 

### Introduction

- colon cancer 
- right and left tumors are morphologically and genetically distinct (why Q is important)
- the microbiome of healthy people and patients with CRC is distinguishable at the OTU level
- specific keystone species are associated with CRC tumors
- Central Question: do the populations of bacteria in the left and right colon contribute to the development of specific subtypes of CRC? 
- nearly all studies of the gut microbiome have been from shed stool or colonoscopy with bowel prep
- bowel prep changes the mucosal/fecal microbiome and what can be analyzed from it. 
- how does the microbiome vary spatially along the colon? we know that the environment varies dramatically but we dont know how those environmental factors specifically affect bacterial populations

- What data is needed to answer this question? first, need to characterize healthy species that are specific to each location. then we can ask, if/how disruption affects tumorigenesis? so what is needed is paired samples from healthy people that have not undergone bowel prep, from the right/left and lumen/mucosa

- what methods are used to get the data? unprepped colonoscopy 

- What analysis must be applied? - diversity analysis and machine learning to pull out small variations/differences among the noisy data set.

- hypothesis: there are distinct bacteria on the left and right side that can be distinguished by 16S sequencing of stool


### Results

#### Microbial membership and diversity of the proximal and distal colon 
Proximal and distal lumenal and mucosal samples were collected from 20 healthy humans that had not undergone bowel preparation prior to the procedure (Figure 1). Participants also collected stool at home one week prior to the procedure. To characterize the bacterial communities present at these sites, 16S rRNA gene sequencing was performed on extracted DNA from each sample. The relative abundance of each phyla for each site is shown in Figure 2A. This data combines the samples from all participants in the study. Each site is primarily dominated by Firmicutes and Bacteriodetes, consistent with known variability in human microbiome research. ((Is anything statistically significant here? use Doherty's code)). Likewise, samples had varying levels of diversity at each site, irrespective of the individual (Figure 2B). That is, while some individuals had a more diverse right mucosa, some had a more diverse left mucosa, and so on. Thus there was no clear pattern of changes in microbial diversity along the gut axis. 

* 2C: NMDS/ordination of some kind ??? maybe if i calculate inter/intrapersonal variability 

To compare similarity between sides (left or right) or sites (lumen or mucosa), we calculated theta YC distances from OTU abundances and compared distances for all individuals. Again, across all patient samples we saw a range of theta YC distances when comparing sample locations (Figure 3A) and again those ranges did not follow a clear pattern on an individual basis. However, when comparing median distances between the right lumen and mucosa, the left versus right lumen, the left versus right mucosa and the left lumen and mucosa, we found that the right lumen and mucosa were most similar to each other than the other samples (P < 0.005, Wilcoxon, BH adjustment). Next, we calculated thetaYC distances to examine how each sample compared to the home-collected exit stool. Amidst variability between patients, we did identify significantly smaller thetaYc distance between the left lumenal sample and the exit stool (Figure 3B, P < 0.05, Wilcoxon, BH adjustment). Further, there was an even larger difference in the comparisons of the left mucosa to the exit stool, indicating that the mucosa is quite different from the stool as compared to lumen (P < 0.0005, Wilcoxon, BH adjustment). This finding suggests that the contents of the left lumen are most representative of stool at exit, and the microbes remaining on the mucosa are much different. ((mention the other 2 comparisons? "among the four comparisons made..."))

#### Random Forest classification models identify important OTUs on each side
To identify OTUs that are distinct at each biogeographical site, we constructed several Random Forest machine-learning models built on OTU abundances. The first model we built to classify the lumen versus mucosal samples for the right and left sides, independently (Figure 4A). The constructed model used ((Xopt? need to follow up with this too)) features for the left and ((Xopt)) for the right. The models performed very well when classifying these samples (AUC X and Y). The OTUs that were most predictive of each site are identified by their greatest mean decrease in accuracy when removed from the model. For distinguishing the right lumen and mucosa, OTUs from the _Bacteriodes_, _Actinomyces_, _Psuedomonas_ and two OTUs from the _Enterobacteraceae_ genera were differentially abundant (Figure 4B). The model classifying the left lumen and mucosa identified OTUs from _Turicibacter_, _Finegoldia_, _Peptoniphilus_ and two OTUs from the _Anaerococcus_ genera that could distinguish lumen from mucosal samples (Figure 4B). These results indicate that there are fine differences between the different sites of the colon, and that these can be traced down to specific OTUs on each side. 

Next, we built a model to differentiate the left and right microbiome. The model performed best when distinguishing the left versus right mucosa (Figure 5A) compared to the left versus right lumen. The model was able to explain ((X%)) of the variance. OTUs that were differentially abundant between the left and right mucosa included members of the _Porphyromonas_, _Murdochiella_, _Finegoldia_, _Anaerococcus_ and _Peptoniphilus_ genera (Figure 5B). The model was less-effective at classifying the right and left lumen ((AUC)). However it did identify differentially abundant OTUs such as three members of the _Bacteroides_ genus, a _Clostridium IV_ OTU and an _Oscillibacter_ OTU (Figure 5C). This analysis found that some of the same OTUs that are distinct between the mucosa and lumen also drive sidedness- such as _Anaerococcus_ and _Finegoldia_. 


### exit comparison models? other findings- fuso etc 

### Discussion
Here we identified bacterial taxa that are specific to the lumen and mucosa of the right and left human colon from samples collected during unprepared colonoscopy. We found that all locations contained a range of phyla and a range of diversity, but that there was a wide variability between subjects. Pairwise comparisons of each of the sites revealed that the right mucosa and lumen were most similar to each other. Further, comparison of colonoscopy-collected samples with samples collected from stool at home showed that the left lumen is most similar to stool at exit. Random Forest algorithms built on OTU abundances from each sample identified microbes that are particular to each location of the colon. 

Recap of results, interesting discussion points about diversity and previous studies of diversity along the gut axis of healthy people

Discussion of differentially abundant OTUs, how they fit the ecological regions of the gut (schematic?)

Put this stuff into context with Sears PNAS paper 

Fuso found on the right side? 


-what does this answer tell us about the broader field?
This answer told us that there are specific differences of bacteria in each side and that we can use this information to form hypotheses about how specific microbial/ecological variations from the host can affect oncogenesis. It tells us too that sample collection methods are important, and what we have found in unprepped bowels do/do not match published findings. 

-what's next? cancer associated OTUs on one side or the other, compare to previous studies. But really to understand what's going on in the various regions of the gut we need more information than just 16S 


### Acknowledgments 

### Methods
Study participants/IRB

Sample collection

Sample processing/DNA extraction

16S rRNA sequencing 

Statistical analysis/Random Forest 

### Figure Legends


Figure 1

Figure 2

Figure 3

Figure 4

Figure 5 

### References 













