\documentclass[11pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
    \usepackage{xltxtra,xunicode}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
  \newcommand{\euro}{â‚¬}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\ifxetex
  \usepackage[setpagesize=false, % page size defined by xetex
              unicode=false, % unicode breaks when used with xetex
              xetex]{hyperref}
\else
  \usepackage[unicode=true]{hyperref}
\fi
\hypersetup{breaklinks=true,
            bookmarks=true,
            pdfauthor={},
            pdftitle={Machine learning identifies unique taxa differentiating proximal and distal human colonic microbiota},
            colorlinks=true,
            citecolor=blue,
            urlcolor=blue,
            linkcolor=magenta,
            pdfborder={0 0 0}}
\urlstyle{same}  % don't use monospace font for urls
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\setcounter{secnumdepth}{0}

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}
  \title{\textbf{Machine learning identifies unique taxa differentiating proximal
and distal human colonic microbiota}}
  \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
  \author{}
  \preauthor{}\postauthor{}
  \date{}
  \predate{}\postdate{}



\begin{document}

\maketitle


\vspace{35mm}

Running title: Unique taxa differentiate proximal and distal human colon
microbiota

\vspace{35mm}

Kaitlin J. Flynn\textsuperscript{1}, Charles C.
Koumpouras\textsuperscript{1}, Mack T. Ruffin IV\textsuperscript{2},
Danielle Kimberly Turgeon\textsuperscript{3}, and Patrick D.
Schloss\textsuperscript{1\(\dagger\)}

\vspace{35mm}

\(\dagger\) Corresponding author:
\href{mailto:pschloss@umich.edu}{\nolinkurl{pschloss@umich.edu}}

1. Department of Microbiology and Immunology, University of Michigan
Medical School, Ann Arbor, Michigan 48109

2. Pennslyvania State University, Hershey, Pennslyvania ??

3. Department of Internal Medicine, Division of Gastroenterology,
University of Michigan Medical School, Ann Arbor, Michigan

\newpage
\linenumbers

\subsubsection{Abstract}\label{abstract}

Colorectal cancer (CRC) remains a leading cause of death worldwide.
Tumors of the proximal (right) and distal (left) colon are
morpologically and genetically distinct. Previous work from our group
found that microbial dysbiosis is associated with the development of
colorectal cancer tumors in studies of both mice and humans. Analysis of
the fecal microbiota from healthy and CRC patients further revealed
different microbial signatures associated with disease. In this study,
we extended our observations of the fecal microbiome to analysis of the
proximal and distal human colon. We used a two-colonoscope approach on
subjects that had not undergone standard bowel preparation procedure.
This technique allowed us to characterize the native proximal and distal
luminal and mucosal microbiome without prior chemical disruption. 16S
rRNA gene sequencing was performed on proximal and distal mucosal
biopsies, luminal and exit stool for 20 healthy individuals. Diversity
analysis of each location revealed that each site contained a diverse
community, and that a patient's samples were more similar to each other
than to that of other individuals. Since we could not differentiate
sites along the colon based on community structure or community
membership alone, we employed the machine-learning algorithm Random
Forest to identify key species that distinguish biogeographical sites.
Random Forest classification models were built using taxa abundance and
sample location revealed distinct populations that were found in each
location. Peptoniphilus, Anaerococcus, Enterobacteraceae, Pseudomonas
and Actinomyces were most likely to be found in mucosal samples versus
luminal samples (AUC = 0.925). The classification model performed well
(AUC = 0.912) when classifying mucosal samples into proximal or distal
sides, but separating luminal samples from each side proved more
challenging (AUC = 0.755). The left mucosa was found to have high
populations of Finegoldia, Murdochiella and Porphyromonas. Proximal and
distal luminal samples were comprised of many of the same taxa, likely
reflecting the fact that stool moves along the colon from the proximal
to distal end. Finally, comparison of all samples to fecal samples taken
at exit uncovered that the feces were most similar to samples taken from
the left lumen, again reflecting the anatomical structure of the colon.
Taken together, our results have identified distinct bacterial
populations distinct of the proximal and distal colon. Further
investigation of these bacteria may elucidate if and how these groups
contribute to differential oncogenesis processes on the respective sides
of the colon.

\subsubsection{Introduction}\label{introduction}

\begin{itemize}
\item
  colon cancer
\item
  right and left tumors are morphologically and genetically distinct
  (why Q is important)
\item
  the microbiome of healthy people and patients with CRC is
  distinguishable at the OTU level
\item
  specific keystone species are associated with CRC tumors
\item
  Central Question: do the populations of bacteria in the left and right
  colon contribute to the development of specific subtypes of CRC?
\item
  nearly all studies of the gut microbiome have been from shed stool or
  colonoscopy with bowel prep
\item
  bowel prep changes the mucosal/fecal microbiome and what can be
  analyzed from it.
\item
  how does the microbiome vary spatially along the colon? we know that
  the environment varies dramatically but we dont know how those
  environmental factors specifically affect bacterial populations
\item
  What data is needed to answer this question? first, need to
  characterize healthy species that are specific to each location. then
  we can ask, if/how disruption affects tumorigenesis? so what is needed
  is paired samples from healthy people that have not undergone bowel
  prep, from the right/left and lumen/mucosa
\item
  what methods are used to get the data? unprepped colonoscopy
\item
  What analysis must be applied? - diversity analysis and machine
  learning to pull out small variations/differences among the noisy data
  set.
\item
  hypothesis: there are distinct bacteria on the left and right side
  that can be distinguished by 16S sequencing of stool
\end{itemize}

\subsubsection{Results}\label{results}

\begin{itemize}
\item
  what data were obtained? 16S sequencing results for samples taken from
  healthy human colons that had not undergone bowel prep Figure 1:
  sample collection schematic
\item
  what were the results of the analyses? the results of the analyses are
  as follows:
\end{itemize}

\paragraph{Descriptive analysis}\label{descriptive-analysis}

\begin{itemize}
\itemsep1pt\parskip0pt\parsep0pt
\item
  there are similar levels (within error margins) of each phyla for each
  location, but there is noise due to interpatient variability. Overall,
  there is also a similar level of diversity in each location. However,
  each patient has a different pattern of diversity over the locations,
  some are high in R, some are high in L, etc. Figure 2:
\item
  2A: relative abundance by phyla
\item
  2B: simpson diversity
\item
  2C: NMDS/ordination of some kind
\end{itemize}

\subsubsection{Discussion}\label{discussion}

-how did the analyses answer the central question?

-what does this answer tell us about the broader field?

\subsubsection{Acknowledgments}\label{acknowledgments}

\subsubsection{Methods}\label{methods}

Study participants/IRB

Sample collection

Sample processing/DNA extraction

16S rRNA sequencing

Statistical analysis/Random Forest

\subsubsection{Figure Legends}\label{figure-legends}

\end{document}
