---
title: '**Machine learning identifies unique taxa differentiating proximal and distal
  human colonic microbiota**'
fontsize: 11pt
geometry: margin=1.0in
output:
  html_document: default
  pdf_document:
    includes: null
    keep_tex: yes
csl: plos.csl
---
\vspace{35mm}

Running title: Specific taxa differentiate proximal and distal human colon microbiota

\vspace{35mm}


Kaitlin J. Flynn^1^, Charles C. Koumpouras^1^, Mack T. Ruffin IV^2^, Danielle Kimberly Turgeon^3^, and Patrick D. Schloss^1$\dagger$^

\vspace{35mm}

$\dagger$ Corresponding author: pschloss@umich.edu


1\. Department of Microbiology and Immunology, University of Michigan Medical School, Ann Arbor, Michigan 48109

2\. Pennslyvania State University, Hershey, Pennslyvania ??

3\. Department of Internal Medicine, Division of Gastroenterology, University of Michigan Medical School, Ann Arbor, Michigan


\newpage
\linenumbers


### Abstract

Colorectal cancer (CRC) remains a leading cause of death worldwide. Tumors of the proximal (right) and distal (left) colon are morphologically and genetically distinct. Previous work from our group found that microbial dysbiosis is associated with the development of colorectal cancer tumors in studies of both mice and humans. Analysis of the fecal microbiota from healthy and CRC patients further revealed different microbial signatures associated with disease. In this study, we extended our observations of the fecal microbiome to analysis of the proximal and distal human colon. We used a two-colonoscope approach on subjects that had not undergone standard bowel preparation procedure. This technique allowed us to characterize the native proximal and distal luminal and mucosal microbiome without prior chemical disruption. 16S rRNA gene sequencing was performed on proximal and distal mucosal biopsies, luminal and exit stool for 20 healthy individuals. Diversity analysis of each location revealed that each site contained a diverse community, and that a patientâ€™s samples were more similar to each other than to that of other individuals. Since we could not differentiate sites along the colon based on community structure or community membership alone, we employed the machine-learning algorithm Random Forest to identify key species that distinguish biogeographical sites. Random Forest classification models were built using taxa abundance and sample location revealed distinct populations that were found in each location. Peptoniphilus, Anaerococcus, Enterobacteraceae, Pseudomonas and Actinomyces were most likely to be found in mucosal samples versus luminal samples (AUC = 0.925). The classification model performed well (AUC = 0.912) when classifying mucosal samples into proximal or distal sides, but separating luminal samples from each side proved more challenging (AUC = 0.755). The left mucosa was found to have high populations of Finegoldia, Murdochiella and Porphyromonas. Proximal and distal luminal samples were comprised of many of the same taxa, likely reflecting the fact that stool moves along the colon from the proximal to distal end. Finally, comparison of all samples to fecal samples taken at exit uncovered that the feces were most similar to samples taken from the left lumen, again reflecting the anatomical structure of the colon. Taken together, our results have identified distinct bacterial populations distinct of the proximal and distal colon. Further investigation of these bacteria may elucidate if and how these groups contribute to differential oncogenesis processes on the respective sides of the colon. 

### Introduction

Colorectal cancer (CRC) is the second-leading cause of cancer-related deaths in the United States (cite). CRC tumors vary in structure, size, morbidity and symptomology depending upon their geographical location in the colon. Tumors found in the proximal colon are typically flatter, sessile tumors that form along the inside of the colon wall whereas tumors of the distal colon are bulbous and project into the lumen. These features make cancers of the proximal colon less likely to present with bowel symptoms and thus are associated with an increase in morbidity and mortality (cite). 

The human colon contains a number of microhabitats that enrich for different microbial membership at each site. Indeed, a number of studies have demonstrated that microbial populations differ throughout the gastrointestinal tract. 


- the microbiome of healthy people and patients with CRC is distinguishable at the OTU level
- specific keystone species are associated with CRC tumors
- Central Question: do the populations of bacteria in the left and right colon contribute to the development of specific subtypes of CRC? 
- nearly all studies of the gut microbiome have been from shed stool or colonoscopy with bowel prep
- bowel prep changes the mucosal/fecal microbiome and what can be analyzed from it. 
- how does the microbiome vary spatially along the colon? we know that the environment varies dramatically but we dont know how those environmental factors specifically affect bacterial populations

- What data is needed to answer this question? first, need to characterize healthy species that are specific to each location. then we can ask, if/how disruption affects tumorigenesis? so what is needed is paired samples from healthy people that have not undergone bowel prep, from the right/left and lumen/mucosa

this is a really good paper, good for framing intro and discussion : Numerical ecology validates a biogeographical distribution and gender-based effect on mucosa-associated bacteria along the human colon

- what methods are used to get the data? unprepped colonoscopy 

- What analysis must be applied? - diversity analysis and machine learning to pull out small variations/differences among the noisy data set.

- hypothesis: there are distinct bacteria on the left and right side that can be distinguished by 16S sequencing of stool


### Results

#### Microbial membership and diversity of the proximal and distal colon 
Proximal and distal lumenal and mucosal samples were collected from 20 healthy humans that had not undergone bowel preparation prior to the procedure (Figure 1). Participants also collected stool at home one week prior to the procedure. To characterize the bacterial communities present at these sites, 16S rRNA gene sequencing was performed on extracted DNA from each sample. The relative abundance of each phyla for each site is shown in Figure 2A. This data combines the samples from all participants in the study. Each site is primarily dominated by _Firmicutes_ and _Bacteriodetes_, consistent with known variability in human microbiome research. ((Is anything statistically significant here? use Doherty's code)). Likewise, samples had varying levels of diversity at each site, irrespective of the individual (Figure 2B). That is, while some individuals had a more diverse right mucosa, some had a more diverse left mucosa, and so on. Thus there was no clear pattern of changes in microbial diversity along the gut axis. 

To determine if the structure of the community is consistent between locations we constructed an NMDS ordination to examine the distances between each sample. We found that microbial communities do not cluster on an ordination by the left or right sides in either the mucosal or lumenal samples (Figure 2C). Previous human microbiome studies that have sampled multiple sites have found that individuals are typically more similar to themselves than others in the study (cite). To see if this effect was driving the organization of the ordination, we connected communities from the same individual using gray lines on Figure 2C. While some individuals did have samples that clustered near each other (shorter lines), there were just as many individuals who's samples diverged from each other (longer lines). Similar to the diversity metrics, when examining community structure between sites and sides, there is no clear pattern between or within individuals.

To compare similarity between sides (left or right) or sites (lumen or mucosa), we calculated theta YC distances from OTU abundances and compared distances for all individuals. Again, across all patient samples we saw a range of theta YC distances when comparing sample locations (Figure 3A) and again those ranges did not follow a clear pattern on an individual basis. However, when comparing median distances between the right lumen and mucosa, the left versus right lumen, the left versus right mucosa and the left lumen and mucosa, we found that the right lumen and mucosa were most similar to each other than the other samples (P < 0.005, Wilcoxon, BH adjustment). Next, we calculated thetaYC distances to examine how each sample compared to the home-collected exit stool. Amidst variability between patients, we did identify significantly smaller thetaYc distance between the left lumenal sample and the exit stool (Figure 3B, P < 0.05, Wilcoxon, BH adjustment). Further, there was an even larger difference in the comparisons of the left mucosa to the exit stool, indicating that the mucosa is quite different from the stool as compared to lumen (P < 0.0005, Wilcoxon, BH adjustment). This finding suggests that the contents of the left lumen are most representative of stool at exit, and the microbes remaining on the mucosa are much different. ((mention the other 2 comparisons? "among the four comparisons made..."))

#### Random Forest classification models identify important OTUs on each side
To identify OTUs that are distinct at each biogeographical site, we constructed several Random Forest machine-learning models built on OTU abundances. The first model we built to classify the lumen versus mucosal samples for the right and left sides, independently (Figure 4A). The constructed model used ((Xopt? need to follow up with this too)) features for the left and ((Xopt)) for the right. The models performed very well when classifying these samples (AUC X and Y). The OTUs that were most predictive of each site are identified by their greatest mean decrease in accuracy when removed from the model. For distinguishing the right lumen and mucosa, OTUs from the _Bacteriodes_, _Actinomyces_, _Psuedomonas_ and two OTUs from the _Enterobacteraceae_ genera were differentially abundant (Figure 4B). The model classifying the left lumen and mucosa identified OTUs from _Turicibacter_, _Finegoldia_, _Peptoniphilus_ and two OTUs from the _Anaerococcus_ genera that could distinguish lumen from mucosal samples (Figure 4C). These results indicate that there are fine differences between the different sites of the colon, and that these can be traced down to specific OTUs on each side. 

Next, we built a model to differentiate the left and right microbiome. The model performed best when distinguishing the left versus right mucosa (Figure 5A) compared to the left versus right lumen. The model was able to explain ((X%)) of the variance. OTUs that were differentially abundant between the left and right mucosa included members of the _Porphyromonas_, _Murdochiella_, _Finegoldia_, _Anaerococcus_ and _Peptoniphilus_ genera (Figure 5B). The model was less-effective at classifying the right and left lumen ((AUC)). However it did identify differentially abundant OTUs such as three members of the _Bacteroides_ genus, a _Clostridium IV_ OTU and an _Oscillibacter_ OTU (Figure 5C). This analysis found that some of the same OTUs that are distinct between the mucosa and lumen also drive sidedness- such as _Anaerococcus_ and _Finegoldia_. 

### Bacterial OTUs associated with cancer are found in healthy individuals 
Some of the healthy patients have Fusobacterium OTUs. The most frequent sequence associated wtih the _Fusobacterium_ genus was OTU179, which BLASTs to _Fusobacterium nucleatum subspecies animalis_. It is the only OTU to align with a nucleatum strain- the only species of _Fusobacterium_ known to have oncogenic properties and be found on the surfaces of colorectal cancer tumors. ((cite joe and niels work somewhere in here, maybe in the discussion?)). The _Fusobacterium_ positive samples were located on the right and left mucosa and ranged up to 1% of the sample. ((Table or something?)). OTU00152 is _Porphyromonas_ and the most frequent sequence in that OTU aligns to _Porphyromonas asacharolytica_, another bacterium commonly detected and isolated from colorectal tumors. OTU152 was only detected on the left mucosa, and in fact was one of the OTUs the classification model identified as separating left and right sides. Samples that were positive for _P. asacharolytica_ ranged in abundances from 0.01% - 16%. Finally, another commonly cancer-associated species, _Parvimonas micra_ was identified to be most similar to OTU248. Only 4 individuals had this bacteria present in their colons, three in the left mucosa and one in the right mucosa. Thus, potentially oncogenic OTUs can be found in our sample set of 20 healthy individuals. 

### Discussion
Here we identified bacterial taxa that are specific to the lumen and mucosa of the right and left human colon from samples collected during unprepared colonoscopy. We found that all locations contained a range of phyla and a range of diversity, but that there was a wide variability between subjects. Pairwise comparisons of each of the sites revealed that the right mucosa and lumen were most similar to each other. Further, comparison of colonoscopy-collected samples with samples collected from stool at home showed that the left lumen is most similar to stool at exit. Random Forest algorithms built on OTU abundances from each sample identified microbes that are particular to each location of the colon. Finally, we were able to detect some bacterial OTUs associated with colorectal cancer in our healthy patient cohort. Using unprepped colonoscopy and machine learning, we have identified bacterial phyla specific to the healthy proximal and distal human colon. 


Discussion of the phyla present and at what levels? Good refs in the gut biogeography paper - mucosa vs lumen enrichment stuff "significant variability between sample locations less than 1 cm apart, which suggests that mucosal microbial populations occur in patches" there is also patchiness in fecal samples themselves: spatial niche partitioning in faeces. . Patchiness/sampling an issue as well as differences in human age, sex, genotype, adn definitely diet, pH, oxygen, etc. (does this go in the intro?)


We identified a range of Simpson diversity values between patients and between each anatomical site. As this measurement takes both number of species and abundance of species into account, it seems that there is a varied level of each component along the human colon. There was no clear pattern of diversity when comparing patients- some patients had a higher level of diversity in the proximal than the distal colon, some patients had the opposite. An early study of microbial diversity and membership of three human subjects found a varied level of diversity in each subject, but that subjects were overall more similar to themselves than to each other (Eckberg 2005). We next used the thetaYC dissimilarity metric to compare the communities present at each anatomical location to each other. We found that while there was a wide range of similarity of each site within a person, the study participants were ultimately more similar to themselves than all of the other participants. This finding is consistent with numerous studies examining the biogeography of the colon in healthy humans (Eckberg 2005, Aguierre de Carcer, 2015, Zhang et al 2013). 

To get around the noisiness created by a diverse set of samples, we employed machine learning algorithms to identify microbes specific to each side. For each comparison we identified top X OTUs that were strongly predictive of one site or another. Generally, OTUs identified in each location were consistent with known physiological gradients along the gut axis (cite nature review). For instance, the right mucosa harbored mucosa-associated facultative anaerobes such as _Actinomyces_ and _Enterobacteraceae_ and the aerobic _Psuedomonas_ consistent with the highest oxygen regions of the colon (cite). The left mucosa was far more likely to host strictly anaerobic species such as _Porphyromonas_, _Anaerococcus_, _Finegoldia_ and _Peptoniphilus_. The model was less effective at classifying the right and left lumenal contents, probably because the samples are arguably composed of the same material. 

offer another model/schematic?
some of the very high OTUs BLAST to stool residents, like that one bacteriodes (OTU20) - not necessarily anything interesting there. 

We detected _F. nucleatum_ and _P. asacharolytica_ in 7 and 5 of our patients, respectively. These bacteria have been shown to be predictive of colorectal cancer in humans (cite Niel) and have oncogenic properties in cell culture and in mice (cite). Interestingly, while _F. nucleatum_ was found on both sides of the colon, _P. asacharolytica_ was only detected in the left mucosa. Not much is known about the distribution of _P. asacharolytica_ but given its documented anaerobic characteristics, it might not be surprising that it resides in the less-oygen-rich left bowel (cite something). In studies examining bacteria on colorectal cancer tumors, _F. nucleatum_ is more commonly detected on right-sided tumors, and distribution of _F. nucleatum_ decreases along the colon to rectum (cite Mima 2016). Of the (8) (40%) individuals positive for _F. nucleatum_ in this study, the bacteria was spread across the right mucosa, left lumen and left mucosa. That said, in a previous study of 1,102 colorectal cancer cases, only 138 (11%) of them contained detectable _F. nucleatum_ at all (Mima 2016). In a separate study of patients with inflammatory bowel disease (IBD), 50% of 22 IBD patients and 17.6% of 34 people without disease contained _F. nucleatum_ in mucosal biopsies (Strauss 2011). These studies have the added caveat of being sampled from prepared or cleaned colons. Thus, the presence of _F. nucleatum_ in a healthy individual is not necessarily linked to the development of future colorectal cancers. Because of the spatial distribution of the _F. nucleatum_ in our sample set, we cannot develop a model for the role of _F. nucleatum_ in the healthy colon. Indeed, rather than the _F. nucleatum_ initiating tumorigenesis, these bacteria may instead arrive "late" to the tumor microenvironment and feed on tumors that have alread begun to develop (Flynn 2016). 

what is rate of colonization with these bugs in healthy in literature? how did these bugs factor into our model? 


Put this stuff into context with Sears PNAS paper on sidedness 



how does bowel prep fit into all of this

Random Forest models have been used by our group and others to increase the detection sensitivity of CRC tumors. Indeed, our group found that a classification model that used microbiome data in combination with Fecal Immunohistochemical Test (FIT) results could correctly identify both carcinoma and adenoma lesions from communities of stool at exit (Baxter et al 2016) and it performed much better than FIT alone. 
- Could try to predict L vs R tumors from exit stool? long term. maybe. probably not. value of getting data about exit stool comparisons. real issue is diagnosing right though. 
- note that we have FIT for all of these people but they're all below limit of detection (except for 2). when we look at models for CRC, cant predict R vs L, so create models for healthy. can also cite niels stuff showing that FIT is representative of stool community. could it also be representative of left? 
- LINK TO EXIT STOOL COMPARISONS.



-what does this answer tell us about the broader field?
This answer told us that there are specific differences of bacteria in each side and that we can use this information to form hypotheses about how specific microbial/ecological variations from the host can affect oncogenesis. It tells us too that sample collection methods are important, and what we have found in unprepped bowels do/do not match published findings. 

-what's next?  But really to understand what's going on in the various regions of the gut we need more information than just 16S 
discuss host response papers? what does that mean. link bugs to host response on left and right? future directions or intro 

### Acknowledgments 

### Methods
Study participants/IRB

Sample collection

Sample processing/DNA extraction

16S rRNA sequencing 

Statistical analysis/Random Forest 

### Figure Legends


Figure 1

Figure 2

Figure 3

Figure 4

Figure 5 

### References 













