\documentclass[11pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdftitle={Unprepared colonoscopy identifies microbiota specific to the proximal and distal human colon},
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}
  \title{Unprepared colonoscopy identifies microbiota specific to the proximal
and distal human colon}
  \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
  \author{}
  \preauthor{}\postauthor{}
  \date{}
  \predate{}\postdate{}

\usepackage{setspace}
\doublespacing
\usepackage{lineno}
\linenumbers

\begin{document}
\maketitle

\vspace{35mm}

Running title: Unprepared colonoscopy identifies proximal and distal
human colonic microbiota

\vspace{35mm}

Kaitlin J. Flynn\textsuperscript{1}, Charles C.
Koumpouras\textsuperscript{1}, Mack T. Ruffin IV\textsuperscript{2},
Danielle Kimberly Turgeon\textsuperscript{3}, and Patrick D.
Schloss\textsuperscript{1\(\dagger\)}

\vspace{35mm}

\(\dagger\) Corresponding author:
\href{mailto:pschloss@umich.edu}{\nolinkurl{pschloss@umich.edu}}

1. Department of Microbiology and Immunology, University of Michigan
Medical School, Ann Arbor, Michigan 48109

2. Pennslyvania State University, Hershey, Pennslyvania ??

3. Department of Internal Medicine, Division of Gastroenterology,
University of Michigan Medical School, Ann Arbor, Michigan

\newpage

\begin{verbatim}
## Warning in roc.default(response, m[[predictors]], ...): 'response' has
## more than two levels. Consider setting 'levels' explicitly or using
## 'multiclass.roc' instead
\end{verbatim}

\subsubsection{Abstract}\label{abstract}

The human colon contains chemicals and nutrients that change along the
proximal to distal gut axis. These gradients create microenvironments
that affect the distrubution and composition of the gut microbiota. The
microbiome has been implicated in the colonic diseases colorectal cancer
(CRC) and inflammatory bowel disease (IBD). Further, these diseases
exhibit different symptoms depending on the location of the colon they
are found in. CRC tumors of the proximal and distal colon are
morphologically and genetically distinct. Similarly, bowel diseases such
as Crohn's are typically exacerbated in the proximal instestine while
ulcerative colitis patients often experience symptoms in the distal
colon. Previous analysis of the fecal microbiota from healthy and CRC or
IBD patients has revealed different microbial signatures associated with
these diseases. We extended these observations of the fecal microbiome
to include analysis of the proximal and distal healthy human colon. We
used a two-colonoscope approach on subjects that had not undergone
standard bowel preparation procedure. This technique allowed us to
characterize the native proximal and distal luminal and mucosal
microbiome without prior chemical disruption. 16S rRNA gene sequencing
was performed on proximal and distal mucosal and luminal biopsies and
home-collected stool for 20 healthy individuals. Diversity analysis
revealed that each site contained a diverse community, and that a
patient's samples were more similar to each other than to that of other
individuals. Comparison of all samples to fecal samples taken at exit
uncovered that the feces were most similar to samples taken from the
distal lumen, likely reflecting the anatomical structure of the colon.
Since we could not differentiate sites along the colon based on
community structure or community membership alone, we employed the
Random Forest machine-learning algorithm to identify key species that
distinguish biogeographical sites. Random Forest classification models
were built using taxa abundance and sample location and revealed
distinct populations that were found in each location.
\emph{Peptoniphilus, Anaerococcus, Enterobacteraceae, Pseudomonas} and
\emph{Actinomyces} were most likely to be found in mucosal samples
versus luminal samples (AUC = 0.925). The classification model performed
well (AUC = 0.912) when classifying mucosal samples into proximal or
distal sides, but separating luminal samples from each side proved more
challenging (AUC = 0.755). The distal mucosa was found to have high
populations of \emph{Finegoldia, Murdochiella} and \emph{Porphyromonas}.
Proximal and distal luminal samples were comprised of many of the same
taxa, likely reflecting the fact that stool moves along the colon from
the proximal to distal end. By sampling the unprepped human colon, our
results have identified distinct bacterial populations native to the
proximal and distal sides. Further investigation of these bacteria may
elucidate if and how these groups contribute to different pathogenesis
processes on the respective sides of the colon.

\subsubsection{Introduction}\label{introduction}

The human colon is an ecosystem comprised of several different
microenvironments inhabited by resident bacterial members of the
microbiome. Concentrations of oxygen, water and anti-microbial peptides
change along the gut axis and influence what populations of microbes
reside in each location. Microenvironments differ not only
longitutinally along the colon, but latitudinally from the epithelium to
mucosa to intestinal lumen, offering several sites for different
microbial communities to flourish.

\emph{2 sentences in here are broken}

The severity, symptoms, morbidity and mortality of these diseases is
known to vary based upon the biogeographical location in which they
occur. For instance, CRC tumors that arise on the distal side of the
colon are infiltrating lesions that present with painful symptoms ((1)).
In contrast, 47\% of CRCs are caused by proximal-sided colon tumors that
are sessile and form along the wall of the colon, often remaining
asymptomatic until advanced carcinogenesis ((1)). The distal and
proximal sides of the colon differ in the amount of inflammation present
and the genomic instability of precancerous cells, respectively, in
addition to variation in the previously mentioned chemical gradients
((2), (3), (4)). In IBD patients, disease flares in the distal colon are
usually indicative of ulcerative colitis (UC), whereas Crohn's disease
(CD) patients typically experience disease in the small intestine, ileum
and proximal colon ((5)). UC presents as large and highly inflammed
mucosal ulcers, where as CD lesions are often smaller and have areas of
normal tissue distributed amongst the flare ((5)). Thus, given the
varied physiology of the proximal-distal axis of the colon and known
differences in disease patterns at these sites, symbiotic microbes and
their metabolites likely vary as well, and may influence the
heterogenous disease prognoses of IBD and CRC. Because CRC is a
long-term complication of IBD, the distribution of microbes is important
to understanding the pathophysiology of both diseases.

Several recent findings have shown that development and progression of
IBD or CRC can be attributed to specific molecular events as a result of
interactions between the gut microbiota and human host ((2)).

\emph{missing last half of second paragraph here, hasnt been fully
tested}

The few existing profiles of the microbial biogeography of the gut have
been limited by sample collection methods. The majority of human gut
microbiome studies have been performed on whole shed stool or on samples
collected during colonoscopy procedures. While the latter method allows
investigators to acquire samples from inside the human colon, typically
this procedure is preceded by the use of bowel preparation methods such
as the consumption of laxatives to cleanse the bowel. Bowel preparation
is essential for detecting cancerous or precancerous lesions in the
colon, but complicates microbiome profiling as the chemicals strip the
bowel of contents and disrupt the mucosal layer ((6), (7)). As such,
what little information we do have about the biogeographical
distribution of the microbes in the proximal and distal colon is
confounded by the bowel preparation procedure.

Here we aimed to address the limitations of previous studies and
effectively identify the microbes specific to the lumen and mucosa of
the proximal and distal healthy human colon. Our design used an
unprepared colonoscopy technique to sample the natural community of each
location of the gut without prior disruption of the native bacteria in
20 healthy volunteers. To address the inherent inter-individual
variation in human microbiomes, we used a machine-learning
classification algorithm trained on curated 16S rRNA sequencing reads to
identify microbes specific to each location. We found that our
classification models were able to separate mucosal and lumenal samples
as well as differentiate between sides of the colon based on populations
of specific microbes. By identifying the specific microbes we are poised
to ask if and how the presence or disruption of the microbes at each
site contribute to the development of the specific tumor subtypes of CRC
in the proximal and distal human colon.

\subsubsection{Results}\label{results}

\paragraph{Microbial membership and diversity of the proximal and distal
colon}\label{microbial-membership-and-diversity-of-the-proximal-and-distal-colon}

Lumenal and mucosal samples were collected from the proximal and distal
colon of 20 healthy humans that had not undergone bowel preparation
(Figure 1). Participants also collected stool at home one week prior to
the procedure. To characterize the bacterial communities present at
these sites, 16S rRNA gene sequencing was performed on extracted DNA
from each sample. Each site was primarily dominated by \emph{Firmicutes}
and \emph{Bacteriodetes} (Figure 2A), consistent with known variability
in human microbiome research (cite). Likewise, samples had varying
levels of diversity at each site, irrespective of the individual (Figure
2B). For example, the proximal mucosa was more diverse than the distal
for some individuals while the opposite was true for others. Therefore
we could not identify a clear pattern of changes in microbial diversity
along the gut axis.

To compare similarity between sides (proximal or distal) or sites (lumen
or mucosa), we calculated \({\theta}\)YC distances from OTU abundances
and compared these distances for all individuals. Again, across all
patient samples we observed a range of \({\theta}\)YC distances when
comparing sample locations (Figure 3A) and again those ranges did not
follow a clear pattern on an individual basis. However, when comparing
median distances between the proximal lumen and mucosa, the proximal
versus distal lumen, the proximal versus distal mucosa, and the distal
lumen and mucosa, we found that the proximal lumen and mucosa were most
similar to each other than the other samples (P \textless{} 0.005,
Wilcoxon, BH adjustment).

\paragraph{Stool at exit most resembles lumenal samples from the distal
colon}\label{stool-at-exit-most-resembles-lumenal-samples-from-the-distal-colon}

Next, we calculated \({\theta}\)YC distances to examine how each sample
compared to the home-collected exit stool. Amidst variability between
patients, we did identify significantly smaller \({\theta}\)YC distance
between the distal lumenal sample and the exit stool (Figure 3B, P
\textless{} 0.05, Wilcoxon, BH adjustment). Furthermore, there was an
even larger difference in the comparisons of the distal mucosa to the
exit stool, indicating that the mucosa is different from the stool as
compared to lumen (P \textless{} 0.0005, Wilcoxon, BH adjustment). To
determine what factors may be driving the differences seen among the
samples, we compared thetaYC distances between samples from all subjects
(interpersonal) versus samples from within one subject (intrapersonal).
We found that samples from one individual were far more similar to each
other than to other study subjects (Figure 3C), consistent with previous
human microbiome studies that have sampled multiple sites of the human
colon (\textbf{???}, (8), (9)). Thus interpersonal variation between
subjects drives the differences between samples more than sample site or
location. Overall, the results comparing the structure of the
communities suggest that the contents of the distal lumen are most
representative of stool at exit, and the microbes remaining on the
mucosa are much different.

\paragraph{Random Forest classification models identify important OTUs
on each
side}\label{random-forest-classification-models-identify-important-otus-on-each-side}

To identify OTUs that were distinct at each biogeographical site, we
constructed several Random Forest models trained using OTU abundances.
We built the first model to classify the lumen versus mucosal samples
for the proximal and distal sides, independently (Figure 4A). The
constructed model used ((Xopt)) features for the proximal and ((Xopt))
for the distal. The models performed well when classifying these samples
(0.8312912 and 0.9804871, respectively). The OTUs that were most
predictive of each site are identified by their greatest mean decrease
in accuracy when removed from the model. For distinguishing the proximal
lumen and mucosa, OTUs from the \emph{Bacteriodes}, \emph{Actinomyces},
\emph{Psuedomonas} and two OTUs from the \emph{Enterobacteraceae} genera
were differentially abundant (Figure 4B). The model classifying the
distal lumen and mucosa identified OTUs from \emph{Turicibacter},
\emph{Finegoldia}, \emph{Peptoniphilus} and two OTUs from the
\emph{Anaerococcus} genera that could distinguish lumen from mucosal
samples (Figure 4C). These results indicate that there are fine
differences between the different sites of the colon, and that these can
be traced down to specific OTUs on each side.

Next, we built a model to differentiate the proximal and distal lumenal
samples. The model performed best when distinguishing the proximal
versus distal mucosa (Figure 5A, AUC = 0.912) compared to the proximal
versus distal lumen (AUC = 0.755). These models were able to explain
((X\%)) of the variance, respectively. OTUs that were differentially
abundant between the distal and proximal mucosa included members of the
\emph{Porphyromonas}, \emph{Murdochiella}, \emph{Finegoldia},
\emph{Anaerococcus} and \emph{Peptoniphilus} genera (Figure 5B).
Differentially abundant OTUs of the proximal and distal lumen included
three OTUs of the \emph{Bacteroides} genus, a \emph{Clostridium IV} OTU
and an \emph{Oscillibacter} OTU (Figure 5C). This analysis found that
some of the same OTUs that are distinct between the mucosa and lumen
also helped to differentiate between the two sides- such as
\emph{Anaerococcus} and \emph{Finegoldia}.

\paragraph{Bacterial OTUs associated with cancer are found in healthy
individuals}\label{bacterial-otus-associated-with-cancer-are-found-in-healthy-individuals}

Given that specific bacterial species have been associated with
colorectal cancer and IBD, we probed our sample set for these OTUs.
Among our 100 samples, the most frequent sequence associated with the
\emph{Fusobacterium} genus was OTU179, which aligns via BLASTn to
\emph{Fusobacterium nucleatum subsp animalis} (XX\% over full length).
This is the only species of \emph{Fusobacterium} known to have oncogenic
properties and be found on the surfaces of colorectal cancer tumors.
((10)). The \emph{Fusobacterium} positive samples were located in x\% of
the the proximal and X\% of the distal mucosa and represented as much as
1\% of any sample (Figure 6A). OTU152 was similar to the members of the
\emph{Porphyromonas} genus and the most frequent sequence in that OTU
aligned to \emph{Porphyromonas asacharolytica} (X\% over full length),
another bacterium commonly detected and isolated from colorectal tumors.
OTU152 was only detected on the distal mucosa, and in fact was one of
the OTUs the classification model identified as separating distal and
proximal sides (Figure 6B). Among the samples that were positive for
\emph{P. asacharolytica}, the relative abundances for this OTU ranged
from 0.01\% - 16\%. Thus, disease-associated OTUs could be found in our
sample set of 20 healthy individuals.

\subsubsection{Discussion}\label{discussion}

Here we identified bacterial taxa that were specific to the lumen and
mucosa of the proximal and distal sides of the human colon from samples
collected during unprepared colonoscopy. We found that all locations
contained a range of phyla and a range of diversity, but that there was
a wide variability between subjects. Pairwise comparisons of each of the
sites revealed that the proximal mucosa and lumen were most similar to
each other. Further, comparison of colonoscopy-collected samples with
samples collected from stool at home showed that the distal lumen is
most similar to stool at exit. Random Forest models built on OTU
relative abundances from each sample identified microbes that are
particular to each location of the colon. Finally, we were able to
detect some bacterial OTUs associated with colonic disease in our
healthy patient cohort. Using unprepped colonoscopy and machine
learning, we have identified bacterial phyla specific to the healthy
proximal and distal human colon.

When examining the relative abundance of the different phyla at each
site, there was a wide amount of variation for each phyla with
communities primarily dominated by the \emph{Bacteriodes} and
\emph{Firmicutes}. This likely reflects not only the variability between
human subjects, casued by differences in age, gender, diet, but also
reports of biogeogrphical ``patchiness'' in the gut microbiome. Several
studies have noted that the bacteria recoverable from the same mucosal
sample location can be vastly different when the samples are taken just
1 cm away from each other (11). Similar patchiness is also observed in
lumenal contents and fecal samples themselves; there is observed
separation of different interacting microbes along the length of a stool
sample, for instance (12). That said, across our samples the mucosal
samples harbor more \emph{Proteobacteria}, consistent with previous
studies comparing mucosal swabs to lumenal content in humans (3). Hence,
the conclusions we can draw from phyla analysis are likely impacted by
patchiness between subjects.

To get around the noisiness from a diverse set of samples, we built a
Random Forest model to identify microbes specific to each side. For each
comparison we identified top X OTUs that were strongly predictive of one
site or another. Generally, OTUs identified in each location were
consistent with known physiological gradients along the gut axis (4).
For instance, the proximal mucosa contains the highest oxygen
concentratons of the colon and harbored mucosa-associated facultative
anaerobes such as \emph{Actinomyces} and \emph{Enterobacteraceae} and
aerobic \emph{Psuedomonas}. The distal mucosa was far more likely to
host strictly anaerobic species such as \emph{Porphyromonas},
\emph{Anaerococcus}, \emph{Finegoldia} and \emph{Peptoniphilus}. The
model was less effective at classifying the proximal and distal lumenal
contents, probably because the samples are arguably composed of the same
bacteria but differ in water content.

We detected \emph{F. nucleatum} and \emph{P. asacharolytica} in 8 and 5
of our subjects, respectively. These bacteria have been shown to be
predictive of colorectal cancer in humans (13) and have oncogenic
properties in cell culture and in mice (14). Interestingly, while
\emph{F. nucleatum} was found on both sides of the colon, \emph{P.
asacharolytica} was only detected in the distal mucosa. Not much is
known about the distribution of \emph{P. asacharolytica} but given its
documented anaerobic characteristics and asacharolytic metabolism, it
might not be surprising that it resides in the less-oygen-rich and
proteinaceous distal mucosa ((3)). In studies examining bacteria on
colorectal cancer tumors, \emph{F. nucleatum} is more commonly detected
on proximal-sided tumors, and distribution of \emph{F. nucleatum}
decreases along the colon to rectum ((15)). Of the 8 (40\%) individuals
positive for \emph{F. nucleatum} in this study, the bacterium was spread
across the proximal mucosa, distal lumen and distal mucosa. The presence
of \emph{F. nucleatum} in a healthy individual is not necessarily linked
to the development of future colorectal cancers (cite). Data examining
bacterial biofilms on CRC tumors suggests that \emph{Fusobacteria}
species are more commonly found on proximal tumors and in biofilms,
indicating that it is not only the presence of the bacteria but the
organization of the tumor community that contributes to
\emph{Fusobacterium's} role in tumorigenesis ((16)). Finally,
\emph{Fusobacterium} and \emph{Porphyromonas} species have been known to
not only co-occur on CRC tumors but also to synergistically promote
tumorigenesis in an oral cancer model ((17), (18)). Thus, further
analysis of the distribution and activities of these pathogens may
elucidate a mechanism for development of IBD or CRC subtypes in the
proximal or distal colon.

The \emph{Fusobacterium} species \emph{nucleatum} and \emph{varium} have
been commonly isolated from mucosal biopsies of patients with IBD
(cite). Laboratory experiments with these isolates have shown that
disease-isolated \emph{F. nucleatum} are more invasive and stimulate
more TNF-a production than strains from healthy individuals ((19)),
suggesting the bacteria may increase inflammation in the gut as well
(cite - Dharmani 2011). \emph{F. varium} isolated from UC patients
caused colonic ulcers in an experimental mouse model (Ohkusa 2003).
\emph{F. varium} was only detected in 3 of our study participants and
two of those samples were isolated form the proximal mucosa (cite
figure). Although there is low occurrance in our study, \emph{F. varium}
is most commonly isolated from UC patient biopsies from the ileum or
cecum (cite), suggesting this species may exhibit preference for the
different environmental conditions of these gastrointestinal sites.
Further work will assess how gut environment may select for species
which then cause localized disease.

Specific comparisons of our findings to previously published gut
biogeography studies are additionally confounded by the use of bowel
preparation methods in most other studies. A rare report of a
matched-colonoscopy study sampled 18 patient's colonic mucosa and
lumenal contents prior to and after bowel cleansing ((20)). This group
found that mucosal and lumenal samples were distinguishable prior to
bowel cleansing, but that bowel preparation resulted in an increase in
shared OTUs between each site ((20)). Bowel cleansing not only made the
samples harder to distinguish, it resulted in decreases in diversity
across sites. Further, the differences were not great enough to overcome
interpersonal differences between subjects. Bowel preparation clearly
induces bias into the microbes recovered from sampling the lumen or
mucosa of a prepared bowel. Thus our findings in this study are
strengthened by the lack of bowel preparation.

By revealing specific differences in microbial populations at each
location in the gut via sampling an unprepared bowel, we can begin to
form hypothesies about how specific host-microbe interactions can affect
disease progression of proximal and distal CRC and IBD subtypes. To this
point, 16S rRNA gene sequecing community profiling studies do not
provide enough information to fully probe these questions. In
particular, 16S sequencing cannot not profile the host characteristics
at each site. Combining the unprepared colonoscopy approach with
analysis of multi-omic sequencing data may be useful in further
characterizing host-microbiome interactions along the gut axis for both
health and disease.

\subsubsection{Acknowledgments}\label{acknowledgments}

We thank all the individuals who volunteered for the study. This work
was supported by the Rose and Lawrence C. Page Foundation (DKT). We
would also like to thank Brian Kleiner, Chelsea Crofoot, and Kirk Herman
for their roles in study coordination, subject recruitment, sample
collection and sample processing.

\subsubsection{Methods}\label{methods}

\paragraph{Human subjects}\label{human-subjects}

The procedures in this study and consent were approved by the
Institutional Review Board at the University of Michigan Health System
with protocol number XXXX. Subjects were recruited using the online
recruitment platform and were pre-screened prior to enrollment in the
study. Exclusion criteria included: use of asprin or NSAIDs within 7
days, use of antibiotics within 3 months, current use of anticoagulants,
known allergies to Fentanyl or Benadryl, prior history of colon disease,
diabetes, abdominal surgery, respiratory, liver, kidney or brain
impairments, undergoing current chemotherapy or radiation treatment and
subjects that were pregnant or trying to conceive. 20 subjects that met
the criteria were selected and provided signed informed consent prior to
the procedure. There were 13 female and 7 male subjects ranging in age
from 25 to 64.

\paragraph{Sample collection}\label{sample-collection}

At a baseline visit, subjects were consented and given a home collection
stool kit (Source of kit supplies). At least one week prior to the
scheduled colonoscopy, subjects were to collect whole stool at home and
ship the samples to the University on ice. Notably, subjects did not
undergo any bowel preparation method prior to sampling. On the procedure
day, subjects reported to the Michigan Clinical Research Unit at the
University of Michigan Health System. Patients were consciously sedated
using Fentanyl, Versed and/or Benadryl as appropriate. A flexible
sigmoidoscope was first inserted about 25cm into the colon and endoscopy
brush used to collect lumenal/stool contents. Two lumenal samples were
collected and the contents immediately deposited into RNAlater (source)
and flash-frozen in liquid nitrogen. The brushes were withdrawn and
biopsy forceps were used to collect mucosal biopsies on sections of the
colon that were pink and free of stool matter. Three mucosal biopsies
were collected and flash-frozen in RNAlater. These samples comprised the
distal or distal colon samples. The sigmoidoscope was then withdrawn and
a pediatric colonoscope was inserted to reach the ascending colon.
Samples were then collected as in the distal colon and the colonoscope
withdrawn. All samples were stored at -80 C until study completion.

\paragraph{Sample processing, sequencing and
analysis}\label{sample-processing-sequencing-and-analysis}

DNA extraction was performed using the PowerMicrobiome DNA/RNA Isolation
Kit (MO BIO Laboratories). For tissue biopsies, Bond-Breaker TCEP
solution (Fisher) and 2.8mm ceramic beads (MO BIO Laboratories) were
added to the bead beating step to enhance DNA recovery from mucosal
samples. The resulting DNA was used as template for amplification of the
V4 region of the 16S rRNA gene and fragments were sequenced on an
Illumina MiSeq as previously described ((21)). Sequences were curated
using the mothur software as described previously ((22)). The sequences
were assigned taxonomic classification using a naive Bayesian classifier
trained using a 16S rRNA gene training set from the Ribosomal Database
Project (RDP) ((23)) and clustered into operational taxonomic units
(OTUs) based on a 97\% similarity cutoff. Sequencing and analysis of a
mock community revealed the error rate to be X\%. Samples were rarefied
to 4231 sequences per sample in order to reduce uneven sampling bias.

Diversity analysis was performed using the Simpson diversity calculator
and theta YC calculator metrics in mothur version
\textless{}3.2?\textgreater{} ((22)). ThetaYC distances were calculated
to determine the dissimilarity between two samples. Random Forest
classification models were built using the randomForest R package and
resultant models were used to identify the OTUs that were most important
for classifying each location ((24)). To get species-level information
about sequences of interest, sequences were aligned using blastn and the
species name was only used if the identity score was \textgreater{}=
99\%.

\paragraph{Statistical analysis}\label{statistical-analysis}

Differences in community membership at the phyla level were tested using
the analysis of molecular variance (AMOVA) metric in mothur. Differences
in thetaYC distances by location were tested using the Wilcoxon rank-sum
test adjusted for multiple comparisons using the Benjamini-Hochberg
procedure.

\paragraph{Data availability}\label{data-availability}

\paragraph{Github repository}\label{github-repository}

\newpage

\subsubsection{Figures}\label{figures}

\paragraph{Figure 1}\label{figure-1}

Sampling strategy. A flexible sigmoidoscope was used to sample the
distal colonic luminal contents and mucosa. The scope was inserted
\textasciitilde{} 25cm into the subject and endoscopy brushes were used
to sample the luminal contents (green star). A separate set of biopsy
forceps was used to sample the distal mucosa (blue star). The
sigmoidoscope was removed. A pediatric colonoscope was inserted and used
to access the proximal colon. Biopsies were taken of the proximal
luminal contents and mucosa as described. One week prior to the
procedure stool was collected at home and sent into the laboratory.
Representative images from one individual are shown.

\newpage

\paragraph{Figure 2}\label{figure-2}

Microbial membership and diversity of the proximal and distal human
colon. A) Relative abundance of the top five bacterial phyla in each
sampling site. Each box represents the median and confidence intervals.
B) Simpson diversity of the microbial communities at each location. The
lines represent the median values.

\newpage

\paragraph{Figure 3}\label{figure-3}

Distances of microbial community structure between sites of the gut.
ThetaYC distances are shown for interpersonal similarities between two
sites -- each point represents one individual. In (A), comparisons of
the proximal and distal mucosal and lumen are shown. In (B), comparisons
of each site to the exit stool are shown.

\newpage

\paragraph{Figure 4}\label{figure-4}

Random Forest classifies the mucosa and lumen of each side of the colon.
A) Receiver Operator Characteristic curves are shown for the 10-fold
cross validation of the Random Forest model classifying lumen and
mucosal samples for the distal and proximal sides of the colon. (B) Top
five OTUs that are most important for the classification model for the
distal mucosa and lumen (B) and the proximal mucosa and lumen (C).

\newpage

\paragraph{Figure 5}\label{figure-5}

Random Forest classifies the distal and proximal sides of the colon. A)
Receiver Operator Characteristic curves are shown for the 10-fold cross
validation of the Random Forest model classifying distal lumen versus
proximal lumen (orange) and distal mucosa vs proximal mucosa (green).
(B) Top five OTUs that are most important for the classification model
for the distal and proximal mucosa (B) and the distal and proximal lumen
(C).

\newpage

\paragraph{Figure 6}\label{figure-6}

Location and abundance of cancer-associated OTUs. Relative abundance was
calculated and plotted by sample site for each OTU of interest: (A)
\emph{Fusobacterium nucleatum} and (B) \emph{Porphyromonas
asacharolytica}

\subsubsection{References}\label{references}

\subsection{this is file from commit right before broken
commit}\label{this-is-file-from-commit-right-before-broken-commit}

\section*{testing intro}\label{testing-intro}
\addcontentsline{toc}{section}{testing intro}

\hypertarget{refs}{}
\hypertarget{ref-Benedix2010}{}
1. \textbf{Benedix F}, \textbf{Kube R}, \textbf{Meyer F},
\textbf{Schmidt U}, \textbf{Gastinger I}, \textbf{Lippert H}. 2010.
Comparison of 17, 641 patients with right- and left-sided colon cancer:
Differences in epidemiology, perioperative course, histology, and
survival. Diseases of the Colon \& Rectum \textbf{53}:57--64.
doi:\href{https://doi.org/10.1007/dcr.0b013e3181c703a4}{10.1007/dcr.0b013e3181c703a4}.

\hypertarget{ref-Yamauchi2012}{}
2. \textbf{Yamauchi M}, \textbf{Lochhead P}, \textbf{Morikawa T},
\textbf{Huttenhower C}, \textbf{Chan AT}, \textbf{Giovannucci E},
\textbf{Fuchs C}, \textbf{Ogino S}. 2012. Colorectal cancer: A tale of
two sides or a continuum?: Figure 1. Gut \textbf{61}:794--797.
doi:\href{https://doi.org/10.1136/gutjnl-2012-302014}{10.1136/gutjnl-2012-302014}.

\hypertarget{ref-Albenberg2014}{}
3. \textbf{Albenberg L}, \textbf{Esipova TV}, \textbf{Judge CP},
\textbf{Bittinger K}, \textbf{Chen J}, \textbf{Laughlin A},
\textbf{Grunberg S}, \textbf{Baldassano RN}, \textbf{Lewis JD},
\textbf{Li H}, \textbf{Thom SR}, \textbf{Bushman FD}, \textbf{Vinogradov
SA}, \textbf{Wu GD}. 2014. Correlation between intraluminal oxygen
gradient and radial partitioning of intestinal microbiota.
Gastroenterology \textbf{147}:1055--1063.e8.
doi:\href{https://doi.org/10.1053/j.gastro.2014.07.020}{10.1053/j.gastro.2014.07.020}.

\hypertarget{ref-Donaldson2015}{}
4. \textbf{Donaldson GP}, \textbf{Lee SM}, \textbf{Mazmanian SK}. 2015.
Gut biogeography of the bacterial microbiota. Nature Reviews
Microbiology \textbf{14}:20--32.
doi:\href{https://doi.org/10.1038/nrmicro3552}{10.1038/nrmicro3552}.

\hypertarget{ref-Forbes2016}{}
5. \textbf{Forbes JD}, \textbf{Domselaar GV}, \textbf{Bernstein CN}.
2016. The gut microbiota in immune-mediated inflammatory diseases.
Frontiers in Microbiology \textbf{7}.
doi:\href{https://doi.org/10.3389/fmicb.2016.01081}{10.3389/fmicb.2016.01081}.

\hypertarget{ref-Jalanka2014}{}
6. \textbf{Jalanka J}, \textbf{Salonen A}, \textbf{Salojärvi J},
\textbf{Ritari J}, \textbf{Immonen O}, \textbf{Marciani L},
\textbf{Gowland P}, \textbf{Hoad C}, \textbf{Garsed K}, \textbf{Lam C},
\textbf{Palva A}, \textbf{Spiller RC}, \textbf{Vos WM de}. 2014. Effects
of bowel cleansing on the intestinal microbiota. Gut
\textbf{64}:1562--1568.
doi:\href{https://doi.org/10.1136/gutjnl-2014-307240}{10.1136/gutjnl-2014-307240}.

\hypertarget{ref-Harrell2012}{}
7. \textbf{Harrell L}, \textbf{Wang Y}, \textbf{Antonopoulos D},
\textbf{Young V}, \textbf{Lichtenstein L}, \textbf{Huang Y},
\textbf{Hanauer S}, \textbf{Chang E}. 2012. Standard colonic lavage
alters the natural state of mucosal-associated microbiota in the human
colon. PLoS ONE \textbf{7}:e32545.
doi:\href{https://doi.org/10.1371/journal.pone.0032545}{10.1371/journal.pone.0032545}.

\hypertarget{ref-deCarcer2010}{}
8. \textbf{Cárcer DA de}, \textbf{Cuív PÓ}, \textbf{Wang T},
\textbf{Kang S}, \textbf{Worthley D}, \textbf{Whitehall V},
\textbf{Gordon I}, \textbf{McSweeney C}, \textbf{Leggett B},
\textbf{Morrison M}. 2010. Numerical ecology validates a biogeographical
distribution and gender-based effect on mucosa-associated bacteria along
the human colon. The ISME Journal \textbf{5}:801--809.
doi:\href{https://doi.org/10.1038/ismej.2010.177}{10.1038/ismej.2010.177}.

\hypertarget{ref-Zhang2013}{}
9. \textbf{Zhang Z}, \textbf{Geng J}, \textbf{Tang X}, \textbf{Fan H},
\textbf{Xu J}, \textbf{Wen X}, \textbf{Ma Z (Sam)}, \textbf{Shi P}.
2013. Spatial heterogeneity and co-occurrence patterns of human
mucosal-associated intestinal microbiota. The ISME Journal
\textbf{8}:881--893.
doi:\href{https://doi.org/10.1038/ismej.2013.185}{10.1038/ismej.2013.185}.

\hypertarget{ref-Castellarin2011}{}
10. \textbf{Castellarin M}, \textbf{Warren RL}, \textbf{Freeman JD},
\textbf{Dreolini L}, \textbf{Krzywinski M}, \textbf{Strauss J},
\textbf{Barnes R}, \textbf{Watson P}, \textbf{Allen-Vercoe E},
\textbf{Moore RA}, \textbf{Holt RA}. 2011. Fusobacterium nucleatum
infection is prevalent in human colorectal carcinoma. Genome Research
\textbf{22}:299--306.
doi:\href{https://doi.org/10.1101/gr.126516.111}{10.1101/gr.126516.111}.

\hypertarget{ref-Hong2011}{}
11. \textbf{Hong P-Y}, \textbf{Croix JA}, \textbf{Greenberg E},
\textbf{Gaskins HR}, \textbf{Mackie RI}. 2011. Pyrosequencing-based
analysis of the mucosal microbiota in healthy individuals reveals
ubiquitous bacterial groups and micro-heterogeneity. PLoS ONE
\textbf{6}:e25042.
doi:\href{https://doi.org/10.1371/journal.pone.0025042}{10.1371/journal.pone.0025042}.

\hypertarget{ref-Stearns2011}{}
12. \textbf{Stearns JC}, \textbf{Lynch MDJ}, \textbf{Senadheera DB},
\textbf{Tenenbaum HC}, \textbf{Goldberg MB}, \textbf{Cvitkovitch DG},
\textbf{Croitoru K}, \textbf{Moreno-Hagelsieb G}, \textbf{Neufeld JD}.
2011. Bacterial biogeography of the human digestive tract. Scientific
Reports \textbf{1}.
doi:\href{https://doi.org/10.1038/srep00170}{10.1038/srep00170}.

\hypertarget{ref-Baxter2016}{}
13. \textbf{Baxter NT}, \textbf{Ruffin MT}, \textbf{Rogers MAM},
\textbf{Schloss PD}. 2016. Microbiota-based model improves the
sensitivity of fecal immunochemical test for detecting colonic lesions.
Genome Medicine \textbf{8}.
doi:\href{https://doi.org/10.1186/s13073-016-0290-3}{10.1186/s13073-016-0290-3}.

\hypertarget{ref-Sears2014}{}
14. \textbf{Sears CL}, \textbf{Garrett WS}. 2014. Microbes, microbiota,
and colon cancer. Cell Host \& Microbe \textbf{15}:317--328.
doi:\href{https://doi.org/10.1016/j.chom.2014.02.007}{10.1016/j.chom.2014.02.007}.

\hypertarget{ref-Mima2016}{}
15. \textbf{Mima K}, \textbf{Cao Y}, \textbf{Chan AT}, \textbf{Qian ZR},
\textbf{Nowak JA}, \textbf{Masugi Y}, \textbf{Shi Y}, \textbf{Song M},
\textbf{Silva A da}, \textbf{Gu M}, \textbf{Li W}, \textbf{Hamada T},
\textbf{Kosumi K}, \textbf{Hanyuda A}, \textbf{Liu L}, \textbf{Kostic
AD}, \textbf{Giannakis M}, \textbf{Bullman S}, \textbf{Brennan CA},
\textbf{Milner DA}, \textbf{Baba H}, \textbf{Garraway LA},
\textbf{Meyerhardt JA}, \textbf{Garrett WS}, \textbf{Huttenhower C},
\textbf{Meyerson M}, \textbf{Giovannucci EL}, \textbf{Fuchs CS},
\textbf{Nishihara R}, \textbf{Ogino S}. 2016. Fusobacterium nucleatum in
colorectal carcinoma tissue according to tumor location. Clinical and
Translational Gastroenterology \textbf{7}:e200.
doi:\href{https://doi.org/10.1038/ctg.2016.53}{10.1038/ctg.2016.53}.

\hypertarget{ref-Dejea2014}{}
16. \textbf{Dejea CM}, \textbf{Wick EC}, \textbf{Hechenbleikner EM},
\textbf{White JR}, \textbf{Welch JLM}, \textbf{Rossetti BJ},
\textbf{Peterson SN}, \textbf{Snesrud EC}, \textbf{Borisy GG},
\textbf{Lazarev M}, \textbf{Stein E}, \textbf{Vadivelu J},
\textbf{Roslani AC}, \textbf{Malik AA}, \textbf{Wanyiri JW}, \textbf{Goh
KL}, \textbf{Thevambiga I}, \textbf{Fu K}, \textbf{Wan F}, \textbf{Llosa
N}, \textbf{Housseau F}, \textbf{Romans K}, \textbf{Wu X},
\textbf{McAllister FM}, \textbf{Wu S}, \textbf{Vogelstein B},
\textbf{Kinzler KW}, \textbf{Pardoll DM}, \textbf{Sears CL}. 2014.
Microbiota organization is a distinct feature of proximal colorectal
cancers. Proceedings of the National Academy of Sciences
\textbf{111}:18321--18326.
doi:\href{https://doi.org/10.1073/pnas.1406199111}{10.1073/pnas.1406199111}.

\hypertarget{ref-Whitmore2014}{}
17. \textbf{Whitmore SE}, \textbf{Lamont RJ}. 2014. Oral bacteria and
cancer. PLoS Pathogens \textbf{10}:e1003933.
doi:\href{https://doi.org/10.1371/journal.ppat.1003933}{10.1371/journal.ppat.1003933}.

\hypertarget{ref-Flynn2016}{}
18. \textbf{Flynn KJ}, \textbf{Baxter NT}, \textbf{Schloss PD}. 2016.
Metabolic and community synergy of oral bacteria in colorectal cancer.
mSphere \textbf{1}:e00102--16.
doi:\href{https://doi.org/10.1128/msphere.00102-16}{10.1128/msphere.00102-16}.

\hypertarget{ref-Strauss2011}{}
19. \textbf{Strauss J}, \textbf{Kaplan GG}, \textbf{Beck PL},
\textbf{Rioux K}, \textbf{Panaccione R}, \textbf{DeVinney R},
\textbf{Lynch T}, \textbf{Allen-Vercoe E}. 2011. Invasive potential of
gut mucosa-derived fusobacterium nucleatum positively correlates with
IBD status of the host. Inflammatory Bowel Diseases
\textbf{17}:1971--1978.
doi:\href{https://doi.org/10.1002/ibd.21606}{10.1002/ibd.21606}.

\hypertarget{ref-Shobar2016}{}
20. \textbf{Shobar RM}, \textbf{Velineni S}, \textbf{Keshavarzian A},
\textbf{Swanson G}, \textbf{DeMeo MT}, \textbf{Melson JE},
\textbf{Losurdo J}, \textbf{Engen PA}, \textbf{Sun Y}, \textbf{Koenig
L}, \textbf{Mutlu EA}. 2016. The effects of bowel preparation on
microbiota-related metrics differ in health and in inflammatory bowel
disease and for the mucosal and luminal microbiota compartments.
Clinical and Translational Gastroenterology \textbf{7}:e143.
doi:\href{https://doi.org/10.1038/ctg.2015.54}{10.1038/ctg.2015.54}.

\hypertarget{ref-Kozich2013}{}
21. \textbf{Kozich JJ}, \textbf{Westcott SL}, \textbf{Baxter NT},
\textbf{Highlander SK}, \textbf{Schloss PD}. 2013. Development of a
dual-index sequencing strategy and curation pipeline for analyzing
amplicon sequence data on the MiSeq illumina sequencing platform.
Applied and Environmental Microbiology \textbf{79}:5112--5120.
doi:\href{https://doi.org/10.1128/aem.01043-13}{10.1128/aem.01043-13}.

\hypertarget{ref-Schloss2009}{}
22. \textbf{Schloss PD}, \textbf{Westcott SL}, \textbf{Ryabin T},
\textbf{Hall JR}, \textbf{Hartmann M}, \textbf{Hollister EB},
\textbf{Lesniewski RA}, \textbf{Oakley BB}, \textbf{Parks DH},
\textbf{Robinson CJ}, \textbf{Sahl JW}, \textbf{Stres B},
\textbf{Thallinger GG}, \textbf{Horn DJV}, \textbf{Weber CF}. 2009.
Introducing mothur: Open-source, platform-independent,
community-supported software for describing and comparing microbial
communities. Applied and Environmental Microbiology
\textbf{75}:7537--7541.
doi:\href{https://doi.org/10.1128/aem.01541-09}{10.1128/aem.01541-09}.

\hypertarget{ref-Wang2007}{}
23. \textbf{Wang Q}, \textbf{Garrity GM}, \textbf{Tiedje JM},
\textbf{Cole JR}. 2007. Naive bayesian classifier for rapid assignment
of rRNA sequences into the new bacterial taxonomy. Applied and
Environmental Microbiology \textbf{73}:5261--5267.
doi:\href{https://doi.org/10.1128/aem.00062-07}{10.1128/aem.00062-07}.

\hypertarget{ref-Liaw2002}{}
24. \textbf{Liaw A}, \textbf{Wiener M}. 2002. Classification and
regression by randomForest. R News: The Newsletter of the R Project
\textbf{2}:18--22.


\end{document}
